{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "!sudo apt-get update\n",
        "!sudo apt-get install samtools\n",
        "!echo 'export PATH=/path/to/samtools:$PATH' >> ~/.bashrc\n",
        "!source ~/.bashrc\n",
        "\n",
        "!apt-get install -y build-essential\n",
        "!apt-get install -y zlib1g-dev\n",
        "!apt-get install -y perl\n",
        "!apt-get install -y cmake\n",
        "!apt-get install -y cpanminus\n",
        "!apt-get install -y sra-toolkit\n",
        "!vdb-config -v\n",
        "!cpanm install URI::Escape\n",
        "!apt-get install -y jellyfish\n",
        "!apt-get install -y bowtie2\n",
        "!pip install -y salmon\n",
        "\n",
        "!rm /usr/local/lib/python3.10/dist-packages/logan/importer.py\n",
        "!rm /usr/local/lib/python3.10/dist-packages/logan/runner.py\n",
        "\n",
        "!wget -O importer.py https://github.com/400lbhacker/biotrinityfixgooglecolab/raw/main/importer.py\n",
        "!mv importer.py /usr/local/lib/python3.10/dist-packages/logan/importer.py\n",
        "\n",
        "!wget -O importer.py https://github.com/400lbhacker/biotrinityfixgooglecolab/raw/main/runner.py\n",
        "!mv importer.py /usr/local/lib/python3.10/dist-packages/logan/runner.py\n",
        "\n",
        "os.environ['PATH'] += \":/usr/bin/bowtie2\"\n",
        "\n",
        "\n",
        "# Download and install Trinity\n",
        "!wget https://github.com/trinityrnaseq/trinityrnaseq/releases/download/v2.12.0/trinityrnaseq-v2.12.0.FULL.tar.gz\n",
        "!tar -xzvf trinityrnaseq-v2.12.0.FULL.tar.gz\n",
        "!rm trinityrnaseq-v2.12.0.FULL.tar.gz\n",
        "!mv trinityrnaseq-v2.12.0/ /usr/local/\n",
        "!echo 'export PATH=\"/usr/local/trinityrnaseq-v2.12.0:$PATH\"' >> ~/.bashrc\n",
        "!source ~/.bashrc\n",
        "\n",
        "# Verify Trinity installation\n",
        "!which Trinity\n",
        "\n",
        "os.chdir('/usr/local/trinityrnaseq-v2.12.0/')\n",
        "!make\n",
        "\n",
        "!samtools --version\n",
        "!bowtie2 --version\n",
        "!salmon --version\n",
        "\n",
        "!rm Trinity\n",
        "!wget -O Trinity https://github.com/400lbhacker/biotrinityfixgooglecolab/blob/main/Trinity\n",
        "#!mv Trinity /usr/local/lib/python3.10/dist-packages/logan/runner.py\n",
        "\n",
        "!/usr/local/trinityrnaseq-v2.12.0/Trinity"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9gCcEAInLlYt",
        "outputId": "a23f4381-c869-457b-bf4d-9a7d664bd045"
      },
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\r0% [Working]\r            \rGet:1 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]\n",
            "\r0% [Waiting for headers] [1 InRelease 13.8 kB/110 kB 12%] [Connected to cloud.r\r                                                                               \rHit:2 http://archive.ubuntu.com/ubuntu jammy InRelease\n",
            "\r0% [1 InRelease 36.3 kB/110 kB 33%] [Waiting for headers] [Connecting to ppa.la\r                                                                               \rGet:3 https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/ InRelease [3,626 B]\n",
            "\r                                                                               \rHit:4 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  InRelease\n",
            "Get:5 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [119 kB]\n",
            "Get:6 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [109 kB]\n",
            "Hit:7 https://ppa.launchpadcontent.net/c2d4u.team/c2d4u4.0+/ubuntu jammy InRelease\n",
            "Get:8 http://security.ubuntu.com/ubuntu jammy-security/main amd64 Packages [915 kB]\n",
            "Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [1,185 kB]\n",
            "Get:10 http://security.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [995 kB]\n",
            "Get:11 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [1,015 kB]\n",
            "Get:12 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1,251 kB]\n",
            "Hit:13 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease\n",
            "Get:14 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy InRelease [24.3 kB]\n",
            "Hit:15 https://ppa.launchpadcontent.net/ubuntugis/ppa/ubuntu jammy InRelease\n",
            "Get:16 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy/main amd64 Packages [37.3 kB]\n",
            "Fetched 5,765 kB in 3s (2,177 kB/s)\n",
            "Reading package lists... Done\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  libhts3 libhtscodecs2\n",
            "Suggested packages:\n",
            "  cwltool\n",
            "The following NEW packages will be installed:\n",
            "  libhts3 libhtscodecs2 samtools\n",
            "0 upgraded, 3 newly installed, 0 to remove and 16 not upgraded.\n",
            "Need to get 963 kB of archives.\n",
            "After this operation, 2,270 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhtscodecs2 amd64 1.1.1-3 [53.2 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libhts3 amd64 1.13+ds-2build1 [390 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu jammy/universe amd64 samtools amd64 1.13-4 [520 kB]\n",
            "Fetched 963 kB in 0s (3,048 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 78, <> line 3.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package libhtscodecs2:amd64.\n",
            "(Reading database ... 122410 files and directories currently installed.)\n",
            "Preparing to unpack .../libhtscodecs2_1.1.1-3_amd64.deb ...\n",
            "Unpacking libhtscodecs2:amd64 (1.1.1-3) ...\n",
            "Selecting previously unselected package libhts3:amd64.\n",
            "Preparing to unpack .../libhts3_1.13+ds-2build1_amd64.deb ...\n",
            "Unpacking libhts3:amd64 (1.13+ds-2build1) ...\n",
            "Selecting previously unselected package samtools.\n",
            "Preparing to unpack .../samtools_1.13-4_amd64.deb ...\n",
            "Unpacking samtools (1.13-4) ...\n",
            "Setting up libhtscodecs2:amd64 (1.1.1-3) ...\n",
            "Setting up libhts3:amd64 (1.13+ds-2build1) ...\n",
            "Setting up samtools (1.13-4) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Processing triggers for libc-bin (2.35-0ubuntu3.1) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n",
            "\n",
            "samtools 1.13\n",
            "Using htslib 1.13+ds\n",
            "Copyright (C) 2021 Genome Research Ltd.\n",
            "\n",
            "Samtools compilation details:\n",
            "    Features:       build=configure curses=yes \n",
            "    CC:             gcc\n",
            "    CPPFLAGS:       -frelease  -Wdate-time -D_FORTIFY_SOURCE=2\n",
            "    CFLAGS:         -g -O2 -ffile-prefix-map=�BUILDPATH�=. -flto=auto -ffat-lto-objects -fstack-protector-strong -Wformat -Werror=format-security\n",
            "    LDFLAGS:        -Wl,-Bsymbolic-functions -flto=auto -Wl,-z,relro -Wl,-z,now\n",
            "    HTSDIR:         \n",
            "    LIBS:           \n",
            "    CURSES_LIB:     -lcurses\n",
            "\n",
            "HTSlib compilation details:\n",
            "    Features:       build=configure plugins=yes, plugin-path=/usr/local/lib/htslib:/usr/local/libexec/htslib:: libcurl=yes S3=yes GCS=yes libdeflate=yes lzma=yes bzip2=yes htscodecs=1.1.1\n",
            "    CC:             gcc\n",
            "    CPPFLAGS:       -I. -DSAMTOOLS=1 -Wdate-time -D_FORTIFY_SOURCE=2\n",
            "    CFLAGS:         -g -O2 -ffile-prefix-map=/build/htslib-TQtOKr/htslib-1.13+ds=. -flto=auto -ffat-lto-objects -fstack-protector-strong -Wformat -Werror=format-security -flto -fvisibility=hidden -flto -fvisibility=hidden\n",
            "    LDFLAGS:        -Wl,-Bsymbolic-functions -flto=auto -Wl,-z,relro -Wl,-z,now -Wl,-flto -fvisibility=hidden -Wl,-flto -fvisibility=hidden\n",
            "\n",
            "HTSlib URL scheme handlers present:\n",
            "    built-in:\t preload, data, file\n",
            "    crypt4gh-needed:\t crypt4gh\n",
            "    mem:\t mem\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "build-essential is already the newest version (12.9ubuntu3).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 16 not upgraded.\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "zlib1g-dev is already the newest version (1:1.2.11.dfsg-2ubuntu9.2).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 16 not upgraded.\n",
            "Reading package lists... Done\n",
            "\n",
            "Reading state information... Done\n",
            "perl is already the newest version (5.34.0-3ubuntu1.2).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 16 not upgraded.\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "cmake is already the newest version (3.22.1-1ubuntu1.22.04.1).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 16 not upgraded.\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "cpanminus is already the newest version (1.7045-1).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 16 not upgraded.\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "sra-toolkit is already the newest version (2.11.3+dfsg-1ubuntu1).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 16 not upgraded.\n",
            "<VdbConfig>\n",
            "<!-- Current configuration -->\n",
            "<Config>\n",
            "  <APPNAME>vdb-config</APPNAME>\n",
            "  <APPPATH>/usr/local/trinityrnaseq-v2.12.0/</APPPATH>\n",
            "  <BUILD>DEBUG</BUILD>\n",
            "  <HOME>/root</HOME>\n",
            "  <HOST/>\n",
            "  <LIBS>\n",
            "    <GUID>806b5c3b-0088-4732-8359-51630950d723</GUID>\n",
            "  </LIBS>\n",
            "  <NCBI_HOME>/root/.ncbi</NCBI_HOME>\n",
            "  <NCBI_SETTINGS>/root/.ncbi/user-settings.mkfg</NCBI_SETTINGS>\n",
            "  <OS>linux</OS>\n",
            "  <PWD>/usr/local/trinityrnaseq-v2.12.0</PWD>\n",
            "  <USER/>\n",
            "  <VDBCOPY>\n",
            "    <DO_NOT_REDACT>CS_KEY,FLOW_CHARS,KEY_SEQUENCE,LINKER_SEQUENCE</DO_NOT_REDACT>\n",
            "    <META>\n",
            "      <IGNORE>col,.seq,STATS,BASE_COUNT,HUFFMAN_TREE_POS,HUFFMAN_TREE_POS_SIZE,HUFFMAN_TREE_PRB,HUFFMAN_TREE_PRB_SIZE,HUFFMAN_TREE_SIG,HUFFMAN_TREE_SIG_SIZE,MSC454_CLIP_QUALITY_LEFT,MSC454_CLIP_QUALITY_RIGHT,MSC454_FLOW_CHARS,MSC454_KEY_SEQUENCE,NREADS,NUMBER_POS_CHANNELS,NUMBER_PRB_CHANNELS_1,NUMBER_PRB_COLUMNS,NUMBER_SIG_CHANNELS,PLATFORM,READ_0,READ_1,SPOT_COUNT</IGNORE>\n",
            "    </META>\n",
            "    <NCBI_SRA_ABI_tbl_v1_1>\n",
            "      <schema>/usr/lib/ncbi-vdb/sra/abi.vschema</schema>\n",
            "      <tab>NCBI:SRA:ABI:tbl:v2</tab>\n",
            "    </NCBI_SRA_ABI_tbl_v1_1>\n",
            "    <NCBI_SRA_Illumina_tbl_v0a_1>\n",
            "      <schema>/usr/lib/ncbi-vdb/sra/illumina.vschema</schema>\n",
            "      <tab>NCBI:SRA:Illumina:tbl:phred:v2</tab>\n",
            "    </NCBI_SRA_Illumina_tbl_v0a_1>\n",
            "    <NCBI_SRA__454__tbl_v0_1>\n",
            "      <schema>sra/454.vschema</schema>\n",
            "      <tab>NCBI:SRA:_454_:tbl:v2</tab>\n",
            "    </NCBI_SRA__454__tbl_v0_1>\n",
            "    <READ_FILTER_COL_NAME>READ_FILTER</READ_FILTER_COL_NAME>\n",
            "    <REDACTABLE_TYPES>INSDC:color:text,INSDC:x2cs:bin,INSDC:2cs:bin,INSDC:2cs:packed,INSDC:dna:text,INSDC:4na:bin,INSDC:4na:packed,INSDC:x2na:bin,INSDC:2na:bin,INSDC:2na:packed,NCBI:SRA:pos16,INSDC:quality:phred,INSDC:quality:log_odds,NCBI:qual4,INSDC:position:one,INSDC:position:zero,NCBI:fsamp4,NCBI:isamp1</REDACTABLE_TYPES>\n",
            "    <REDACTVALUE>\n",
            "      <INSDC_color_text>\n",
            "        <VALUE>'.'</VALUE>\n",
            "      </INSDC_color_text>\n",
            "      <INSDC_dna_text>\n",
            "        <VALUE>'N'</VALUE>\n",
            "      </INSDC_dna_text>\n",
            "      <NCBI_qual4>\n",
            "        <VALUE>-6</VALUE>\n",
            "      </NCBI_qual4>\n",
            "      <TYPES>INSDC_color_text,INSDC_dna_text,NCBI_qual4</TYPES>\n",
            "    </REDACTVALUE>\n",
            "    <SCORE>\n",
            "      <INSDC_2cs_bin>2</INSDC_2cs_bin>\n",
            "      <INSDC_2cs_packed>2</INSDC_2cs_packed>\n",
            "      <INSDC_2na_bin>2</INSDC_2na_bin>\n",
            "      <INSDC_2na_packed>2</INSDC_2na_packed>\n",
            "      <INSDC_SRA_read_type>1</INSDC_SRA_read_type>\n",
            "      <INSDC_quality_log_odds>1</INSDC_quality_log_odds>\n",
            "      <INSDC_quality_phred>1</INSDC_quality_phred>\n",
            "      <INSDC_x2cs_bin>1</INSDC_x2cs_bin>\n",
            "      <INSDC_x2na_bin>1</INSDC_x2na_bin>\n",
            "    </SCORE>\n",
            "    <_454_>\n",
            "      <schema>sra/454.vschema</schema>\n",
            "      <tab>NCBI:SRA:_454_:tbl:v2</tab>\n",
            "    </_454_>\n",
            "    <_ABSOLID_>\n",
            "      <schema>/usr/lib/ncbi-vdb/sra/abi.vschema</schema>\n",
            "      <tab>NCBI:SRA:ABI:tbl:v2</tab>\n",
            "    </_ABSOLID_>\n",
            "    <_ILLUMINA_>\n",
            "      <schema>/usr/lib/ncbi-vdb/sra/illumina.vschema</schema>\n",
            "      <tab>NCBI:SRA:Illumina:tbl:phred:v2</tab>\n",
            "    </_ILLUMINA_>\n",
            "  </VDBCOPY>\n",
            "  <VDB_CONFIG/>\n",
            "  <VDB_ROOT/>\n",
            "  <config>\n",
            "    <default>true</default>\n",
            "  </config>\n",
            "  <kfg>\n",
            "    <arch>\n",
            "      <bits>64</bits>\n",
            "      <name>afa0d13d8cbc</name>\n",
            "    </arch>\n",
            "    <dir>/etc/ncbi</dir>\n",
            "    <name>debian.kfg</name>\n",
            "  </kfg>\n",
            "  <libs>\n",
            "    <cloud>\n",
            "      <report_instance_identity>false</report_instance_identity>\n",
            "    </cloud>\n",
            "  </libs>\n",
            "  <repository>\n",
            "    <remote>\n",
            "      <main>\n",
            "        <CGI>\n",
            "          <resolver-cgi>https://trace.ncbi.nlm.nih.gov/Traces/names/names.fcgi</resolver-cgi>\n",
            "        </CGI>\n",
            "        <SDL.2>\n",
            "          <resolver-cgi>https://locate.ncbi.nlm.nih.gov/sdl/2/retrieve</resolver-cgi>\n",
            "        </SDL.2>\n",
            "      </main>\n",
            "      <protected>\n",
            "        <CGI>\n",
            "          <resolver-cgi>https://trace.ncbi.nlm.nih.gov/Traces/names/names.fcgi</resolver-cgi>\n",
            "        </CGI>\n",
            "        <SDL.2>\n",
            "          <resolver-cgi>https://locate.ncbi.nlm.nih.gov/sdl/2/retrieve</resolver-cgi>\n",
            "        </SDL.2>\n",
            "      </protected>\n",
            "    </remote>\n",
            "    <user>\n",
            "      <ad>\n",
            "        <public>\n",
            "          <apps>\n",
            "            <file>\n",
            "              <volumes>\n",
            "                <flat/>\n",
            "                <flatAd>.</flatAd>\n",
            "              </volumes>\n",
            "            </file>\n",
            "            <refseq>\n",
            "              <volumes>\n",
            "                <refseqAd>.</refseqAd>\n",
            "              </volumes>\n",
            "            </refseq>\n",
            "            <sra>\n",
            "              <volumes>\n",
            "                <sraAd>.</sraAd>\n",
            "              </volumes>\n",
            "            </sra>\n",
            "            <sraPileup>\n",
            "              <volumes>\n",
            "                <ad>.</ad>\n",
            "              </volumes>\n",
            "            </sraPileup>\n",
            "            <sraRealign>\n",
            "              <volumes>\n",
            "                <ad>.</ad>\n",
            "              </volumes>\n",
            "            </sraRealign>\n",
            "            <wgs>\n",
            "              <volumes>\n",
            "                <wgsAd>.</wgsAd>\n",
            "              </volumes>\n",
            "            </wgs>\n",
            "          </apps>\n",
            "          <root>.</root>\n",
            "        </public>\n",
            "      </ad>\n",
            "      <main>\n",
            "        <public>\n",
            "          <apps>\n",
            "            <file>\n",
            "              <volumes>\n",
            "                <flat>files</flat>\n",
            "              </volumes>\n",
            "            </file>\n",
            "            <nakmer>\n",
            "              <volumes>\n",
            "                <nakmerFlat>nannot</nakmerFlat>\n",
            "              </volumes>\n",
            "            </nakmer>\n",
            "            <nannot>\n",
            "              <volumes>\n",
            "                <nannotFlat>nannot</nannotFlat>\n",
            "              </volumes>\n",
            "            </nannot>\n",
            "            <refseq>\n",
            "              <volumes>\n",
            "                <refseq>refseq</refseq>\n",
            "              </volumes>\n",
            "            </refseq>\n",
            "            <sra>\n",
            "              <volumes>\n",
            "                <sraFlat>sra</sraFlat>\n",
            "              </volumes>\n",
            "            </sra>\n",
            "            <sraPileup>\n",
            "              <volumes>\n",
            "                <withExtFlat>sra</withExtFlat>\n",
            "              </volumes>\n",
            "            </sraPileup>\n",
            "            <sraRealign>\n",
            "              <volumes>\n",
            "                <withExtFlat>sra</withExtFlat>\n",
            "              </volumes>\n",
            "            </sraRealign>\n",
            "            <wgs>\n",
            "              <volumes>\n",
            "                <wgsFlat>wgs</wgsFlat>\n",
            "              </volumes>\n",
            "            </wgs>\n",
            "          </apps>\n",
            "        </public>\n",
            "      </main>\n",
            "    </user>\n",
            "  </repository>\n",
            "  <sra>\n",
            "    <quality_type>raw_scores</quality_type>\n",
            "  </sra>\n",
            "  <strings>\n",
            "    <names>https://trace.ncbi.nlm.nih.gov/Traces/names/names.fcgi</names>\n",
            "    <sdl>https://locate.ncbi.nlm.nih.gov/sdl/2/retrieve</sdl>\n",
            "  </strings>\n",
            "  <tls>\n",
            "    <ca.crt>\n",
            "      <ncbi1>-----BEGIN CERTIFICATE-----\n",
            "MIIDrzCCApegAwIBAgIQCDvgVpBCRrGhdWrJWZHHSjANBgkqhkiG9w0BAQUFADBh\n",
            "MQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMRkwFwYDVQQLExB3\n",
            "d3cuZGlnaWNlcnQuY29tMSAwHgYDVQQDExdEaWdpQ2VydCBHbG9iYWwgUm9vdCBD\n",
            "QTAeFw0wNjExMTAwMDAwMDBaFw0zMTExMTAwMDAwMDBaMGExCzAJBgNVBAYTAlVT\n",
            "MRUwEwYDVQQKEwxEaWdpQ2VydCBJbmMxGTAXBgNVBAsTEHd3dy5kaWdpY2VydC5j\n",
            "b20xIDAeBgNVBAMTF0RpZ2lDZXJ0IEdsb2JhbCBSb290IENBMIIBIjANBgkqhkiG\n",
            "9w0BAQEFAAOCAQ8AMIIBCgKCAQEA4jvhEXLeqKTTo1eqUKKPC3eQyaKl7hLOllsB\n",
            "CSDMAZOnTjC3U/dDxGkAV53ijSLdhwZAAIEJzs4bg7/fzTtxRuLWZscFs3YnFo97\n",
            "nh6Vfe63SKMI2tavegw5BmV/Sl0fvBf4q77uKNd0f3p4mVmFaG5cIzJLv07A6Fpt\n",
            "43C/dxC//AH2hdmoRBBYMql1GNXRor5H4idq9Joz+EkIYIvUX7Q6hL+hqkpMfT7P\n",
            "T19sdl6gSzeRntwi5m3OFBqOasv+zbMUZBfHWymeMr/y7vrTC0LUq7dBMtoM1O/4\n",
            "gdW7jVg/tRvoSSiicNoxBN33shbyTApOB6jtSj1etX+jkMOvJwIDAQABo2MwYTAO\n",
            "BgNVHQ8BAf8EBAMCAYYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUA95QNVbR\n",
            "TLtm8KPiGxvDl7I90VUwHwYDVR0jBBgwFoAUA95QNVbRTLtm8KPiGxvDl7I90VUw\n",
            "DQYJKoZIhvcNAQEFBQADggEBAMucN6pIExIK+t1EnE9SsPTfrgT1eXkIoyQY/Esr\n",
            "hMAtudXH/vTBH1jLuG2cenTnmCmrEbXjcKChzUyImZOMkXDiqw8cvpOp/2PV5Adg\n",
            "06O/nVsJ8dWO41P0jmP6P6fbtGbfYmbW0W5BjfIttep3Sp+dWOIrWcBAI+0tKIJF\n",
            "PnlUkiaY4IBIqDfv8NZ5YBberOgOzW6sRBc4L0na4UU+Krk2U886UAb3LujEV0ls\n",
            "YSEY1QSteDwsOoBrp+uvFRTp2InBuThs4pFsiv9kuXclVzDAGySj4dzp30d8tbQk\n",
            "CAUw7C29C79Fv1C5qfPrmAESrciIxpg0X40KPMbp1ZWVbd4wOTAeBggrBgEFBQcD\n",
            "BAYIKwYBBQUHAwEGCCsGAQUFBwMDDBdEaWdpQ2VydCBHbG9iYWwgUm9vdCBDQQ==\n",
            "-----END CERTIFICATE-----\n",
            "</ncbi1>\n",
            "      <ncbi2>-----BEGIN CERTIFICATE-----\n",
            "MIIDxTCCAq2gAwIBAgIQAqxcJmoLQJuPC3nyrkYldzANBgkqhkiG9w0BAQUFADBs\n",
            "MQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMRkwFwYDVQQLExB3\n",
            "d3cuZGlnaWNlcnQuY29tMSswKQYDVQQDEyJEaWdpQ2VydCBIaWdoIEFzc3VyYW5j\n",
            "ZSBFViBSb290IENBMB4XDTA2MTExMDAwMDAwMFoXDTMxMTExMDAwMDAwMFowbDEL\n",
            "MAkGA1UEBhMCVVMxFTATBgNVBAoTDERpZ2lDZXJ0IEluYzEZMBcGA1UECxMQd3d3\n",
            "LmRpZ2ljZXJ0LmNvbTErMCkGA1UEAxMiRGlnaUNlcnQgSGlnaCBBc3N1cmFuY2Ug\n",
            "RVYgUm9vdCBDQTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMbM5XPm\n",
            "+9S75S0tMqbf5YE/yc0lSbZxKsPVlDRnogocsF9ppkCxxLeyj9CYpKlBWTrT3JTW\n",
            "PNt0OKRKzE0lgvdKpVMSOO7zSW1xkX5jtqumX8OkhPhPYlG++MXs2ziS4wblCJEM\n",
            "xChBVfvLWokVfnHoNb9Ncgk9vjo4UFt3MRuNs8ckRZqnrG0AFFoEt7oT61EKmEFB\n",
            "Ik5lYYeBQVCmeVyJ3hlKV9Uu5l0cUyx+mM0aBhakaHPQNAQTXKFx01p8VdteZOE3\n",
            "hzBWBOURtCmAEvF5OYiiAhF8J2a3iLd48soKqDirCmTCv2ZdlYTBoSUeh10aUAsg\n",
            "EsxBu24LUTi4S8sCAwEAAaNjMGEwDgYDVR0PAQH/BAQDAgGGMA8GA1UdEwEB/wQF\n",
            "MAMBAf8wHQYDVR0OBBYEFLE+w2kD+L9HAdSYJhoIAu9jZCvDMB8GA1UdIwQYMBaA\n",
            "FLE+w2kD+L9HAdSYJhoIAu9jZCvDMA0GCSqGSIb3DQEBBQUAA4IBAQAcGgaX3Nec\n",
            "nzyIZgYIVyHbIUf4KmeqvxgydkAQV8GK83rZEWWONfqe/EW1ntlMMUu4kehDLI6z\n",
            "eM7b41N5cdblIZQB2lWHmiRk9opmzN6cN82oNLFpmyPInngiK3BD41VHMWEZ71jF\n",
            "hS9OMPagMRYjyOfiZRYzy78aG6A9+MpeizGLYAiJLQwGXFK3xPkKmNEVX58Svnw2\n",
            "Yzi9RKR/5CYrCsSXaQ3pjOLAEFe4yHYSkVXySGnYvCoCWw9E1CAx2/S6cCZdkGCe\n",
            "vEsXCS+0yx5DaMkHJ8HSXPfqIbloEpw8nL+e/IBcm2PN7EeqJSdnoDfzAIJ9VNep\n",
            "+OkuE6N36B9KMEQwHgYIKwYBBQUHAwQGCCsGAQUFBwMBBggrBgEFBQcDAwwiRGln\n",
            "aUNlcnQgSGlnaCBBc3N1cmFuY2UgRVYgUm9vdCBDQQ==\n",
            "-----END CERTIFICATE-----\n",
            "</ncbi2>\n",
            "    </ca.crt>\n",
            "  </tls>\n",
            "  <tools>\n",
            "    <ascp>\n",
            "      <max_rate>450m</max_rate>\n",
            "    </ascp>\n",
            "  </tools>\n",
            "  <vdb>\n",
            "    <lib>\n",
            "      <paths>\n",
            "        <kfg>/usr/lib/x86_64-linux-gnu</kfg>\n",
            "      </paths>\n",
            "    </lib>\n",
            "  </vdb>\n",
            "</Config>\n",
            "\n",
            "<ConfigurationFiles>\n",
            "/etc/ncbi/certs.kfg\n",
            "/etc/ncbi/debian.kfg\n",
            "/etc/ncbi/default.kfg\n",
            "/etc/ncbi/vdb-copy.kfg\n",
            "</ConfigurationFiles>\n",
            "<Environment>\n",
            "LD_LIBRARY_PATH=/usr/local/nvidia/lib:/usr/local/nvidia/lib64\n",
            "</Environment>\n",
            "</VdbConfig>\n",
            "install is up to date. (0.01)\n",
            "URI::Escape is up to date. (5.21)\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  libjellyfish-2.0-2\n",
            "The following NEW packages will be installed:\n",
            "  jellyfish libjellyfish-2.0-2\n",
            "0 upgraded, 2 newly installed, 0 to remove and 16 not upgraded.\n",
            "Need to get 840 kB of archives.\n",
            "After this operation, 2,463 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libjellyfish-2.0-2 amd64 2.3.0-12ubuntu2 [65.0 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu jammy/universe amd64 jellyfish amd64 2.3.0-12ubuntu2 [775 kB]\n",
            "Fetched 840 kB in 0s (2,236 kB/s)\n",
            "Selecting previously unselected package libjellyfish-2.0-2:amd64.\n",
            "(Reading database ... 122509 files and directories currently installed.)\n",
            "Preparing to unpack .../libjellyfish-2.0-2_2.3.0-12ubuntu2_amd64.deb ...\n",
            "Unpacking libjellyfish-2.0-2:amd64 (2.3.0-12ubuntu2) ...\n",
            "Selecting previously unselected package jellyfish.\n",
            "Preparing to unpack .../jellyfish_2.3.0-12ubuntu2_amd64.deb ...\n",
            "Unpacking jellyfish (2.3.0-12ubuntu2) ...\n",
            "Setting up libjellyfish-2.0-2:amd64 (2.3.0-12ubuntu2) ...\n",
            "Setting up jellyfish (2.3.0-12ubuntu2) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "Processing triggers for libc-bin (2.35-0ubuntu3.1) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n",
            "\n",
            "/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n",
            "\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "bowtie2 is already the newest version (2.4.4-1).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 16 not upgraded.\n",
            "\n",
            "Usage:   \n",
            "  pip3 install [options] <requirement specifier> [package-index-options] ...\n",
            "  pip3 install [options] -r <requirements file> [package-index-options] ...\n",
            "  pip3 install [options] [-e] <vcs project url> ...\n",
            "  pip3 install [options] [-e] <local project path> ...\n",
            "  pip3 install [options] <archive url/path> ...\n",
            "\n",
            "no such option: -y\n",
            "--2023-09-03 00:13:36--  https://github.com/400lbhacker/biotrinityfixgooglecolab/raw/main/importer.py\n",
            "Resolving github.com (github.com)... 140.82.114.4\n",
            "Connecting to github.com (github.com)|140.82.114.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://raw.githubusercontent.com/400lbhacker/biotrinityfixgooglecolab/main/importer.py [following]\n",
            "--2023-09-03 00:13:36--  https://raw.githubusercontent.com/400lbhacker/biotrinityfixgooglecolab/main/importer.py\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 3156 (3.1K) [text/plain]\n",
            "Saving to: ‘importer.py’\n",
            "\n",
            "importer.py         100%[===================>]   3.08K  --.-KB/s    in 0s      \n",
            "\n",
            "2023-09-03 00:13:36 (29.4 MB/s) - ‘importer.py’ saved [3156/3156]\n",
            "\n",
            "--2023-09-03 00:13:36--  https://github.com/400lbhacker/biotrinityfixgooglecolab/raw/main/runner.py\n",
            "Resolving github.com (github.com)... 140.82.112.4\n",
            "Connecting to github.com (github.com)|140.82.112.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://raw.githubusercontent.com/400lbhacker/biotrinityfixgooglecolab/main/runner.py [following]\n",
            "--2023-09-03 00:13:37--  https://raw.githubusercontent.com/400lbhacker/biotrinityfixgooglecolab/main/runner.py\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 5384 (5.3K) [text/plain]\n",
            "Saving to: ‘importer.py’\n",
            "\n",
            "importer.py         100%[===================>]   5.26K  --.-KB/s    in 0s      \n",
            "\n",
            "2023-09-03 00:13:37 (45.9 MB/s) - ‘importer.py’ saved [5384/5384]\n",
            "\n",
            "/usr/bin/bowtie2-align-s version 2.4.4\n",
            "64-bit\n",
            "usage: salmon [--config=/path/to/settings.py] [command] [options]\n",
            "--2023-09-03 00:13:37--  https://github.com/trinityrnaseq/trinityrnaseq/releases/download/v2.12.0/trinityrnaseq-v2.12.0.FULL.tar.gz\n",
            "Resolving github.com (github.com)... 140.82.114.4\n",
            "Connecting to github.com (github.com)|140.82.114.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/22736952/2bfd0500-7d2b-11eb-994a-b0b1d8b37679?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20230903%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230903T001337Z&X-Amz-Expires=300&X-Amz-Signature=df82122d1d0d38d1ee63265722fd895a4b76c1d03164d89d6aae5143e4e2806f&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=22736952&response-content-disposition=attachment%3B%20filename%3Dtrinityrnaseq-v2.12.0.FULL.tar.gz&response-content-type=application%2Foctet-stream [following]\n",
            "--2023-09-03 00:13:37--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/22736952/2bfd0500-7d2b-11eb-994a-b0b1d8b37679?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20230903%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230903T001337Z&X-Amz-Expires=300&X-Amz-Signature=df82122d1d0d38d1ee63265722fd895a4b76c1d03164d89d6aae5143e4e2806f&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=22736952&response-content-disposition=attachment%3B%20filename%3Dtrinityrnaseq-v2.12.0.FULL.tar.gz&response-content-type=application%2Foctet-stream\n",
            "Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
            "Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.108.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 32152192 (31M) [application/octet-stream]\n",
            "Saving to: ‘trinityrnaseq-v2.12.0.FULL.tar.gz’\n",
            "\n",
            "trinityrnaseq-v2.12 100%[===================>]  30.66M   167MB/s    in 0.2s    \n",
            "\n",
            "2023-09-03 00:13:38 (167 MB/s) - ‘trinityrnaseq-v2.12.0.FULL.tar.gz’ saved [32152192/32152192]\n",
            "\n",
            "trinityrnaseq-v2.12.0/\n",
            "trinityrnaseq-v2.12.0/util/\n",
            "trinityrnaseq-v2.12.0/util/abundance_estimates_to_matrix.pl\n",
            "trinityrnaseq-v2.12.0/util/retrieve_sequences_from_fasta.pl\n",
            "trinityrnaseq-v2.12.0/util/analyze_blastPlus_topHit_coverage.pl\n",
            "trinityrnaseq-v2.12.0/util/PBS/\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.p3\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.p1\n",
            "trinityrnaseq-v2.12.0/util/PBS/pbs_check.pl\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.p5b\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.p4b\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.p4a\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.header\n",
            "trinityrnaseq-v2.12.0/util/PBS/README\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_kill.sh\n",
            "trinityrnaseq-v2.12.0/util/PBS/N50stats.pl\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.p2\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.sh\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_pbs.cont\n",
            "trinityrnaseq-v2.12.0/util/PBS/trinity_kill.pl\n",
            "trinityrnaseq-v2.12.0/util/PBS/TRINITY.CONFIG.template\n",
            "trinityrnaseq-v2.12.0/util/R/\n",
            "trinityrnaseq-v2.12.0/util/R/get_Poisson_conf_intervals.R\n",
            "trinityrnaseq-v2.12.0/util/R/expression_analysis_lib.R\n",
            "trinityrnaseq-v2.12.0/util/TrinityStats.pl\n",
            "trinityrnaseq-v2.12.0/util/insilico_read_normalization.pl\n",
            "trinityrnaseq-v2.12.0/util/sift_bam_max_cov.pl\n",
            "trinityrnaseq-v2.12.0/util/filter_low_expr_transcripts.pl\n",
            "trinityrnaseq-v2.12.0/util/align_and_estimate_abundance.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/\n",
            "trinityrnaseq-v2.12.0/util/misc/hicpipe_raw_converter.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/cat_require_newlines.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_bowtie2.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/randomly_mutate_seqs.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_HISAT.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_trimmomatic_qual_trimming.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/count_features_given_MIN_FPKM_threshold.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/omp_iworm_thread_contigs_to_fa.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/collate_fqs.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/insilico_norm_kmer_hists/\n",
            "trinityrnaseq-v2.12.0/util/misc/insilico_norm_kmer_hists/result.pdf\n",
            "trinityrnaseq-v2.12.0/util/misc/insilico_norm_kmer_hists/kmer_histo.all.txt\n",
            "trinityrnaseq-v2.12.0/util/misc/insilico_norm_kmer_hists/kmer_histo.NormMaxKCov50.txt\n",
            "trinityrnaseq-v2.12.0/util/misc/insilico_norm_kmer_hists/plot_me.R\n",
            "trinityrnaseq-v2.12.0/util/misc/kmer_counter.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/illustrate_ref_comparison.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_coordsorted_max_reads_per_position.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gene_to_shared_transcript_content.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/merge_RSEM_output_to_matrix.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gmap_gff3_to_percent_length_stats.count_mapped_transcripts.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastq_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gff3_file_utr_coverage_trimmer.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/alexie_analyze_blast.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/examine_strand_specificity.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/PerlLib/\n",
            "trinityrnaseq-v2.12.0/util/misc/PerlLib/SegmentGraph.pm\n",
            "trinityrnaseq-v2.12.0/util/misc/fastq_unweave_pairs.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/analyze_blastPlus_topHit_coverage.by_prioritized_compreh_category.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/tblastn_wrapper.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_nameSorted_to_uniq_count_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gene_gff3_to_introns.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastQ_append_acc.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_pair_to_bed.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/filter_out_accs_from_fasta.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/HiCpipe_nameSortedSam_to_raw.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_bwasw_trinity.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/plot_strand_specificity_dist_by_quantile.Rscript\n",
            "trinityrnaseq-v2.12.0/util/misc/describe_SAM_read_flag_info.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/print_kmers.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/determine_RF_strand_specificity.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/trinity_component_distribution.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/join_by_left_col.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/examine_weldmer_halves.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/prop_pair_sam_refiner.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/capture_orig_n_unmapped_reads.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_show_alignment.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/decode_SAM_flag_value.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/component_to_graph_dot.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/get_welds_from_chrysals_graphFromFasta_out.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/join_expr_vals_single_table.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/align_reads_launch_igv.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/simulate_illuminaPE_from_transcripts.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gff3_file_to_cdna.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/strip_fasta_header.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gtf_to_introns.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/get_GC_content_dist.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fan_out_fasta_seqs_to_indiv_files.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/plot_expressed_gene_dist.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gmap_gff3_to_percent_length_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gmap_gff3_chimera_jaccard_analyzer.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/transdecoder_pep_to_false_fusion_finder.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sort_fastq.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blast_outfmt6_group_segments.to_Markov_Clustering.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_STAR.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/nameSorted_SAM_to_FastQ.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/plot_ExN50_statistic.Rscript\n",
            "trinityrnaseq-v2.12.0/util/misc/run_read_simulator_per_gene.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_sortAny_to_count_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastQ_top_N_records.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/compare_FL_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/randomly_sample_PE_fastq.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/analyze_blastPlus_topHit_coverage.annotate_details_w_FL_info.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/flattened_gff_n_genome_to_Trinity_emulator.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/transcript_gff3_to_bed.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/filter_similar_seqs_expr_and_strand_aware.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/partition_target_transcripts.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/util/\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/util/find_pruned_edges_shouldve_kept.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/info_files_to_eval_cmds.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/audit_summary_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/write_simulate_read_commands.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/audit_summary_stats.reexamine.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/run_Trinity_eval.sh\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/run_Trinity_eval.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sim_test_framework/run_simulate_reads.wgsim.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/row_to_column.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fasta_filter_by_min_length.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_intron_extractor.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/identify_distal_isoform_variations.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/print.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/Trinity_genome_aligned_gff3_to_regrouped_genes_gtf.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastq_cleaner.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/frag_boundary_to_wig.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/altsplice_simulation_toolkit/\n",
            "trinityrnaseq-v2.12.0/util/misc/altsplice_simulation_toolkit/sim_single_bubble.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastQ_rand_subset.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/allele_simulator.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/get_path_nodes_from_fasta.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/trinity_trans_matrix_to_rep_trans_gene_matrix.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/cdna_fasta_file_to_transcript_gtf.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/combined_nameSorted_to_dup_pairs_removed.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/testUnlimitStacksize.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/sixFrameTranslation.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/mpi_iworm_proc_contigs_to_fa.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/contig_ExN50_statistic.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blast_outfmt6_group_segments.tophit_coverage.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/count_N50_given_MIN_FPKM_threshold.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/simulate_illuminaPE_from_transcripts.wgsim.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/seqinfo_refseq_to_dot.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/organize_data_table_by_trinity_component.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gmap_native_to_format_converter.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/map_gtf_transcripts_to_genome_annots.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/examine_iworm_FL_across_threads.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/jaccard_sam_pair_refiner.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/tab_to_fasta.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fasta_seq_length.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/TPM_weighted_gene_length.py\n",
            "trinityrnaseq-v2.12.0/util/misc/merge_blast_n_rsem_results.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fasta_write_sense_n_anti.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_jellyfish.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/extract_fastQ_pairings.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_HISAT2_via_samples_file.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/transcript_coverage_UTR_trimmer.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/bam_to_cuff.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/cuff_gtf_to_bed.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/sim_reads.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/notes\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/run_trinity_WITH_LR.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/gene_gff3_to_bed_cmds.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/trans_gff3_to_bed_cmds.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/gmap_to_ref.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iso_reco_analysis/run_trinity_no_LR.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_HiCpipe_bowtie.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_GSNAP.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/check_chrysalis_graph_reciprocal_edges.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fasta_to_cmd_generator.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/frag_to_bed.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/m8_blastclust.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/pairwise_kmer_content_comparer.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SRA_to_fastq.notes\n",
            "trinityrnaseq-v2.12.0/util/misc/get_longest_isoform_seq_per_trinity_gene.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gff3_to_genome_feature_base_encoding.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/bam_gene_tests/\n",
            "trinityrnaseq-v2.12.0/util/misc/bam_gene_tests/write_trin_cmds.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/bam_gene_tests/harvest_transcripts.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/bam_gene_tests/extract_bam_reads_per_target_transcript.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/bam_gene_tests/extract_bam_reads_per_target_gene.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/simulate_reads_sam_and_fa.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/count_iso_per_gene_dist.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/transcript_fasta_to_ORF_pics.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blastn_wrapper.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/check_fastQ_pair_ordering.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/Monarch_util/\n",
            "trinityrnaseq-v2.12.0/util/misc/Monarch_util/generate_gene_alt_splicing_graphs.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/Monarch_util/generate_trans_graphs.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/blat_to_sam.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/blat_sam_add_reads2.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/process_BLAT_alignments.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/run_BLAT_shortReads.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/pslx_to_gff3.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/blat_top_hit_extractor.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blat_util/top_blat_sam_extractor.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/count_trans_per_component.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gtf_to_bed_format.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastq_merge_sorted_tab_lists.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_STAR_via_samples_file.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/compare_bflies.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/TophatCufflinksWrapper.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_toString.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/analyze_blastPlus_topHit_coverage.extract_OS.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/Monarch\n",
            "trinityrnaseq-v2.12.0/util/misc/analyze_blastPlus_topHit_coverage.org_matrix.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/nameSorted_SAM_to_paired_fastq.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/ButterflyFastaToGraphDot.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_to_bed.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/pair_up_fastq_files_LeftRight.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastq_interleave_pairs.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_DETONATE.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/alt_GG_read_partitioning_JCornish/\n",
            "trinityrnaseq-v2.12.0/util/misc/alt_GG_read_partitioning_JCornish/genwig2.py\n",
            "trinityrnaseq-v2.12.0/util/misc/alt_GG_read_partitioning_JCornish/genwig.sh\n",
            "trinityrnaseq-v2.12.0/util/misc/splice_path_analysis/\n",
            "trinityrnaseq-v2.12.0/util/misc/splice_path_analysis/assess_intron_path_sensitivity.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/splice_path_analysis/intron_barcharter.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/splice_path_analysis/assess_intron_path_sensitivity.summarizer.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/splice_path_analysis/diff_splice_paths.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/chrys_graph_to_dot.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/iworm_welds_to_dot.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fasta_file_reformatter.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/Artemis/\n",
            "trinityrnaseq-v2.12.0/util/misc/Artemis/join_multi_wig_to_graph_plot.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/cdhit_examine_isoforms.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_to_fasta.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_read_simulator_per_fasta_entry.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/tab_to_fastQ.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/pair_up_fastq_files_1_2.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_bwa.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/remove_cntrl_chars.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SRA_to_fastq.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/join_any.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/blast_outfmt6_group_segments.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/TEST_SUPPORT/\n",
            "trinityrnaseq-v2.12.0/util/misc/TEST_SUPPORT/BFLY_TESTING/\n",
            "trinityrnaseq-v2.12.0/util/misc/TEST_SUPPORT/BFLY_TESTING/graph_out_to_bfly_cmd.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastQ_rand_subset.SE.reservoir_sampling_reqiures_high_mem.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/gff3_to_genome_feature_base_encoding.parse_SAM.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fastQ_rand_subset.reservoir_sampling_reqiures_high_mem.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/ensure_paired_end_bam_file.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/process_GMAP_alignments_gff3_chimeras_ok.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/N50.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/merge_rsem_n_express_for_compare.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/acc_list_to_fasta_entries.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/fasta_remove_duplicates.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/average.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/SAM_show_alignment.summarize_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/run_TOPHAT.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/count_number_fasta_seqs.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/shuffle.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/genome_gff3_to_gene_gff3_partitions.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/pair_up_fastq_files_R1_R2.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/count_matrix_features_given_MIN_TPM_threshold.pl\n",
            "trinityrnaseq-v2.12.0/util/misc/convert_fasta_identifiers_for_FL_analysis.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_to_frag_coords.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/filter_transcripts_require_min_cov.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/fasta_find_duplicates.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/kallisto_trans_to_gene_results.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/bowtie2_wrapper.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_extract_properly_mapped_pairs.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_filter_out_unmapped_reads.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/filter_iworm_by_min_length_or_cov.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_coordSorted_fragment_coverage_writer2.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/get_Trinity_gene_to_trans_map.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/ExitTester.jar\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/partitioned_trinity_aggregator.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/fragment_coverage_writer.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/jaccard_fasta_clipper.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/ensure_coord_sorted_sam.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/join_partitions_within_range.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/plugin_install_tests.sh\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/batch_cmds.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/nbkc_normalize.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/wig_clip_to_bed.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/tests/\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/tests/tests.py\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/tests/test_prep.py\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/tests/sample_data_tests.py\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_ordered_pair_jaccard.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/trinity_install_tests.sh\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/partition_chrysalis_graphs_n_reads.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/trinity_installer.py\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/scaffold_iworm_contigs.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/salmon_trans_to_gene_results.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_coordSorted_fragment_Read_coverage_writer.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/revcomp_fasta.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/fasta_to_tab.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_strand_separator.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/merge_pair_and_LR_scaff_links.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/add_LR_reads_to_iworm_bundle.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/define_SAM_coverage_partitions2.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/nbkc_merge_left_right_stats.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/write_partitioned_trinity_cmds.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/run_TMM_scale_matrix.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/annotate_chrysalis_welds_with_iworm_names.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/run_UpperQuartileNormalization_matrix.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/extract_reads_per_partition.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/fastQ_to_fastA.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/inchworm_transcript_splitter.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/define_coverage_partitions.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/jaccard_wig_clipper.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/prep_rnaseq_alignments_for_genome_assisted_assembly.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/segment_GFF_partitions.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/fastQ_to_tab.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/iworm_LR_to_scaff_pairs.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/eXpress_trans_to_gene_results.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_extract_uniquely_mapped_reads.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/SAM_set_transcribed_orient_info.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/ordered_fragment_coords_to_jaccard.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/GG_partitioned_trinity_aggregator.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/process_GMAP_alignments_gff3_chimeras_ok.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/print_butterfly_assemblies.pl\n",
            "trinityrnaseq-v2.12.0/util/support_scripts/outfmt6_add_percent_match_length.pl\n",
            "trinityrnaseq-v2.12.0/Inchworm/\n",
            "trinityrnaseq-v2.12.0/Inchworm/LICENSE\n",
            "trinityrnaseq-v2.12.0/Inchworm/CMakeLists.txt\n",
            "trinityrnaseq-v2.12.0/Inchworm/Makefile\n",
            "trinityrnaseq-v2.12.0/Inchworm/README\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/MaskedKmerCounter.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Fasta_reader.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/cigar_tweaker.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/cigar_tweaker\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/argProcessor.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/fastaToKmerCoverageStats.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/stacktrace.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/pull_reads_with_kmers.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/SNP_stomper.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/SAM_entry.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/Fasta_reader.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/Cigar.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/cigar_tweaker.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/IRKE.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/IRKE_run.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/DeBruijnGraph.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/SAM_reader.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/string_util.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/KmerCounter.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/FastaToDeBruijn.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/sequenceUtil.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/MaskedKmerCounter.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/.deps/Fasta_entry.Po\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/inchworm\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/KmerCounter.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/fastaToKmerCoverageStats.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/SAM_entry.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/SAM_reader.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/pull_reads_with_kmers\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/DeBruijnGraph.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Cigar.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/stacktrace.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/string_util.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/stacktrace.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/SAM_entry.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/SNP_stomper.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/IRKE_run.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Ktree.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/pull_reads_with_kmers.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/argProcessor.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/KtreeNode.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Ktree.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/string_util.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/argProcessor.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/prev.Makefile.am\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/mpi_deprecated/\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/mpi_deprecated/MPI_InchwormMakefile\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/mpi_deprecated/MPI_cray.Makefile\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/mpi_deprecated/MPIinchworm.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Cigar.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Fasta_entry.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/DeBruijnGraph.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/irke_common.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/SNP_stomper\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/SAM_reader.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/sequenceUtil.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Fasta_entry.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/fastaToKmerCoverageStats\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/FastaToDeBruijn.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/common.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/Fasta_reader.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/MaskedKmerCounter.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/IRKE.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/KmerCounter.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/KtreeNode.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/IRKE.hpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/sequenceUtil.cpp\n",
            "trinityrnaseq-v2.12.0/Inchworm/src/FastaToDeBruijn\n",
            "trinityrnaseq-v2.12.0/PerlLib/\n",
            "trinityrnaseq-v2.12.0/PerlLib/SAM_reader.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/GFF_maker.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Overlap_piler.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/test_Fasta_retriever.pl\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/Splice_graph_assembler.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/PASA_alignment_assembler.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/Gene_obj_alignment_assembler.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/Alternative_splice_comparer.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/CDNA_stitcher.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/Genome_based_cDNA_assembler.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/CDNA_alignment.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/Overlap_assembler.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/Alignment_segment.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CDNA/Genome_based_cDNA_graph_assembler.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/VCF_parser.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Simulate/\n",
            "trinityrnaseq-v2.12.0/PerlLib/Simulate/Uniform_Read_Generator.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/GTF.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Ktree.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Pipeliner.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Thread_helper.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Longest_orf.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/PSL_parser.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/test_htc_gridrunner_LSF.pl\n",
            "trinityrnaseq-v2.12.0/PerlLib/Fasta_retriever.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/TiedHash.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/COMMON.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CanvasXpress/\n",
            "trinityrnaseq-v2.12.0/PerlLib/CanvasXpress/Heatmap.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/EM.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/SAM_entry.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Fasta_reader.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Process_cmd.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Exons_to_geneobj.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/GTF_utils.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CIGAR.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Gene_obj.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/ColorGradient.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/BED_utils.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Ascii_genome_illustrator.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/SingleLinkageClusterer.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Fastq_reader.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/SAM_reader.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/ReadManager.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/SAM_to_AlignGraph.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/StringNode.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/AlignGraph.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/StringGraph.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/SAM_entry.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/KmerNode.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/GenericGraph.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/ReadTracker.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/ReadCoverageNode.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/AlignNode.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/ReadCoverageGraph.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/GenericNode.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/KmerGraphLib/KmerGraph.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Nuc_translator.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Gene_obj_indexer.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/CMD_processor.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/GFF3_utils.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/Overlap_info.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/DelimParser.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/test_htc_gridrunner_SGE.pl\n",
            "trinityrnaseq-v2.12.0/PerlLib/overlapping_nucs.ph\n",
            "trinityrnaseq-v2.12.0/PerlLib/BHStats.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/WigParser.pm\n",
            "trinityrnaseq-v2.12.0/PerlLib/GFF3_alignment_utils.pm\n",
            "trinityrnaseq-v2.12.0/Trinity\n",
            "trinityrnaseq-v2.12.0/README.md\n",
            "trinityrnaseq-v2.12.0/LICENSE\n",
            "trinityrnaseq-v2.12.0/make.macOSX.sh\n",
            "trinityrnaseq-v2.12.0/.gitmodules\n",
            "trinityrnaseq-v2.12.0/PyLib/\n",
            "trinityrnaseq-v2.12.0/PyLib/Pipeliner.py\n",
            "trinityrnaseq-v2.12.0/Changelog.txt\n",
            "trinityrnaseq-v2.12.0/Makefile\n",
            "trinityrnaseq-v2.12.0/README\n",
            "trinityrnaseq-v2.12.0/Chrysalis/\n",
            "trinityrnaseq-v2.12.0/Chrysalis/aligns/\n",
            "trinityrnaseq-v2.12.0/Chrysalis/aligns/KmerAlignCore.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/aligns/KmerAlignCore.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/make.log\n",
            "trinityrnaseq-v2.12.0/Chrysalis/util/\n",
            "trinityrnaseq-v2.12.0/Chrysalis/util/mutil.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/util/checkLock.c\n",
            "trinityrnaseq-v2.12.0/Chrysalis/util/mutil.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/README.md\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/SVector.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/ErrorHandling.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/FileParser.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/StringUtil.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/FileParser.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/CommandLineParser.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/ErrorHandling.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/base/StringUtil.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/chrysalis.notes\n",
            "trinityrnaseq-v2.12.0/Chrysalis/CMakeLists.txt\n",
            "trinityrnaseq-v2.12.0/Chrysalis/system/\n",
            "trinityrnaseq-v2.12.0/Chrysalis/system/BigFileDefines.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/Makefile\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/GraphFromFasta.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/CreateIwormFastaBundle.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/JoinTransByPairs.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/TranscriptomeGraph.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/BubbleUpClustering.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/sequenceUtil.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/string_util.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/DNAVector.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/stacktrace.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/DNAVector.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/QuantifyGraph.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/KmerTable.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/KmerTable.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/NonRedKmerTable.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/DeBruijnGraph.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/stacktrace.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/Chrysalis.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/ReadsToComponents.pl\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/IsoformAugment.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/AACodons.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/DeBruijnGraph.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/BreakTransByPairs.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/TranscriptomeFromVaryK.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/AACodons.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/Pool.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/TranscriptomeGraph.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/ReadsToTranscripts.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/CompMgr.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/sequenceUtil.h\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/NonRedKmerTable.cc\n",
            "trinityrnaseq-v2.12.0/Chrysalis/analysis/RunButterfly.cc\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/include/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/include/cmd_line_opts.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/include/common.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/include/graphnode.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/include/graph.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/README\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/bin/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/bin/.hidden\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/src/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/src/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/src/graph.cpp\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/src/cmd_line_opts.cpp\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/src/graphnode.cpp\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/slclust/src/slcluster.cpp\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/ParaFly-0.1.0.tar.gz\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/scaffold_iworm_contigs/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/scaffold_iworm_contigs/error_checker.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/scaffold_iworm_contigs/error_checker.cpp\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/scaffold_iworm_contigs/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/scaffold_iworm_contigs/ScaffoldIwormContigs.cpp\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/README.md\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hfile_s3.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/bcf_sr_sort.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/thread_pool.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/tbx.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/version.sh\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/plugin.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/sam_internal.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/README.md\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htsfile.1\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/textutils_internal.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/thread_pool_internal.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/textutils.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/LICENSE\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/khash_str2int.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/kfunc.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/faidx.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/khash.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/ksort.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/tbx.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/vcf.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/hts_endian.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/regidx.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/hts.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/hfile.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/vcfutils.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/synced_bcf_reader.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/hts_defs.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/kstring.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/kseq.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/kbitset.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/hts_log.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/vcf_sweep.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/knetfile.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/hts_os.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/sam.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/klist.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/thread_pool.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/cram.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib/bgzf.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hfile_s3_write.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib.pc.in\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib_vars.mk\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hfile_libcurl.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/.appveyor.yml\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/README.large_positions.md\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/range.cram.crai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/no_hdr_sq_1.bam.csi\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn02.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test_bgzf.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn02.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#minimal.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/simple_test_driver.sh\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1#noseq.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/header_syms.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/longrefs/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/longrefs/longref.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/longrefs/index.expected1.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/longrefs/index.expected2.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/longrefs/longref_multi.expected.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/longrefs/longref_itr.expected.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/longrefs/index.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.cram.crai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/formatmissing-out.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thrash_threads1.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#unmap2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#repeated.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn02_exp.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#5b_java.cram\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/bgziptest.txt.gz.gzi\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thrash_threads3.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-vcf-api.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-vcf-hdr-in.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#5b.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#tlen.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-vcf-api.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/gff_file.gff\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/bed_file.Y.100200.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/vcf_file.1.3000151.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/large_chr.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/tabix.tst\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/large_chr.20.1.2147483647.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/bed_file.bed\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/test-tabix.sh\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/vcf_file.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/gff_file.X.2934832.2935190.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix/vcf_file.2.3199812.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/range.bam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#1.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/formatmissing.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.sam.gz.csi\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn01.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/fastqs.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/colons.bam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-regidx.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#tlen2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#unmap.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.bcf.csi\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/noroundtrip-out.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.bam.csi\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/sam.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1#pad3.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1#pad1.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn01.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#large_aux_java.cram\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#large_seq.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/formatcols.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thrash_threads2.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/auxf#values_java.cram\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/range.bam.bai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test_kstring.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#1000.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c2.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1#pad2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/fastqs_README.txt\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/md.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#unsorted.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#5.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1#clip.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#large_aux.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#blank.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn01_exp.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test_realn.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1#bounds.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#tag_padded.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/hfile.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/fuzz/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/fuzz/hts_open_fuzzer.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn01_exp-a.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thread_pool.md\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn01.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/bgziptest.txt.gz\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#pair.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thrash_threads4.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#tag_depadded.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#rg.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/bgziptest.txt\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/auxf#values.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#triplet.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn01_exp-e.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn02_exp-a.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-bcf-translate.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/maintainer/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/maintainer/check_copyright.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/maintainer/check_spaces.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/compare_sam.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/md.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.bam.bai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#large_aux2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-bcf-translate.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/cross_validate.sh\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#unmap1.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-bcf-sr.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/md#1.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/auxf.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-vcf-sweep.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.vcf.gz.tbi\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c2.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thrash_threads6.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/hts_endian.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test_index.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn02-r.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.vcf.gz.csi\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/noroundtrip.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/fieldarith.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thrash_threads5.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c2#pad.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#MD2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/ce#supp.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-vcf-sweep.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/pileup.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test_str2int.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test_view.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.sam.gz.bai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/fieldarith.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/emptyfile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-parse-reg.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/range.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-vcf-hdr.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/auxf.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/faidx.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad1.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mpileup.tst\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/test-pileup.sh\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_D.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_N.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_N2.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_I.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_ID.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_DI.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad3.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad1.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/small.bam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_P.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_ID.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/small.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad2.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_P.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad3.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_I.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_N.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_DI.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_N2.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/mpileup/mp_D.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-bcf-sr.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn02.fa.fai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/range.cram\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/no_hdr_sq_1.expected.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/xx#MD.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/no_hdr_sq_1.bam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/c1#unknown.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/thrash_threads7.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/realn02_exp-e.sam\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/tabix.out\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/index.vcf\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/test-logging.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/test/colons.bam.bai\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/sam.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/INSTALL\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hfile_net.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/bcf_sr_sort.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/multipart.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/configure.ac\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/region.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/tabix.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/sam.5\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_io.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/open_trace_file.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/string_alloc.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/rANS_byte.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/mFILE.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_encode.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_codecs.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_external.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/string_alloc.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/rANS_static.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/open_trace_file.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_samtools.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/rANS_static.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_structs.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_index.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_decode.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/pooled_alloc.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/mFILE.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_stats.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/os.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/pooled_alloc.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_encode.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_index.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_stats.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_codecs.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_samtools.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_decode.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/misc.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/cram/cram_io.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/bgzip.1\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hfile.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/README\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/knetfile.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/bgzf.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib-s3-plugin.7\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/aclocal.m4\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/faidx.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/vcf.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/kstring.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/synced_bcf_reader.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hfile_gcs.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/bgzip.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/header.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hts_internal.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/.gitignore\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hts.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/tabix.1\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/config.mk.in\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hfile_internal.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/os/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/os/lzma_stub.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/os/rand.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/hts_os.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htslib.mk\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/kfunc.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/probaln.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/.travis.yml\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/regidx.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/vcf.5\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/md5.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/NEWS\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/vcf_sweep.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/vcfutils.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/realn.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/m4/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/m4/hts_hide_dynamic_syms.m4\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/m4/hts_prog_cc_warnings.m4\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/errmod.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/header.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/htsfile.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/faidx.5\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/htslib/.gitattributes\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/sift_bam_max_cov.cpp\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/.gitmodules\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/bamsifter\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/bamsifter/build_htslib.sh\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/README\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/BIN/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/BIN/README\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/LICENSE\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/adapters/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/adapters/TruSeq3-SE.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/adapters/TruSeq3-PE.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/adapters/TruSeq2-PE.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/adapters/TruSeq3-PE-2.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/adapters/NexteraPE-PE.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/adapters/TruSeq2-SE.fa\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/trimmomatic.jar\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/Trimmomatic-0.36/trimmomatic-0.36.jar\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/README.md\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/khash.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/LICENSE\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/unequal_seq_n_quals_1.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/oldformat_1.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/corrupt_1.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/oldformat_2.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/newformat_2.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/forward_reverse_notation_2.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/forward_reverse_notation_1.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/newformat_1.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/stripped.fq\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/testing/test_seqtk_trinity.py\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/seqtk.c\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/Makefile\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/kseq.h\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/seqtk-trinity/.gitignore\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/COLLECTL/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/COLLECTL/util/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/COLLECTL/util/collectl_dat_to_time_matrix.py\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/COLLECTL/util/plot_time_vs_resource.Rscript\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/COLLECTL/collectl-4.1.0.src.tar.gz\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/COLLECTL/examine_resource_usage_profiling.pl\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/DEXseq_util/\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/DEXseq_util/dexseq_prepare_annotation.py\n",
            "trinityrnaseq-v2.12.0/trinity-plugins/DEXseq_util/README.txt\n",
            "trinityrnaseq-v2.12.0/Docker/\n",
            "trinityrnaseq-v2.12.0/Docker/VERSION.txt\n",
            "trinityrnaseq-v2.12.0/Docker/cleanMe.sh\n",
            "trinityrnaseq-v2.12.0/Docker/README.md\n",
            "trinityrnaseq-v2.12.0/Docker/push_docker.sh\n",
            "trinityrnaseq-v2.12.0/Docker/run_ext_test_singularity.sh\n",
            "trinityrnaseq-v2.12.0/Docker/make_simg.sh\n",
            "trinityrnaseq-v2.12.0/Docker/run_ext_test_docker.sh\n",
            "trinityrnaseq-v2.12.0/Docker/run_test_docker.sh\n",
            "trinityrnaseq-v2.12.0/Docker/test_data/\n",
            "trinityrnaseq-v2.12.0/Docker/test_data/reads.right.fq.gz\n",
            "trinityrnaseq-v2.12.0/Docker/test_data/reads.left.fq.gz\n",
            "trinityrnaseq-v2.12.0/Docker/build_docker.sh\n",
            "trinityrnaseq-v2.12.0/Docker/Dockerfile\n",
            "trinityrnaseq-v2.12.0/Docker/notes.txt\n",
            "trinityrnaseq-v2.12.0/Docker/.dockerignore\n",
            "trinityrnaseq-v2.12.0/.gitignore\n",
            "trinityrnaseq-v2.12.0/WDL/\n",
            "trinityrnaseq-v2.12.0/WDL/trinity_abundance_estimates_to_matrix.wdl\n",
            "trinityrnaseq-v2.12.0/WDL/trinity_differential_expression.wdl\n",
            "trinityrnaseq-v2.12.0/WDL/build.sh\n",
            "trinityrnaseq-v2.12.0/WDL/parse_commands.py\n",
            "trinityrnaseq-v2.12.0/WDL/trinityrnaseq.wdl\n",
            "trinityrnaseq-v2.12.0/WDL/Dockerfile\n",
            "trinityrnaseq-v2.12.0/WDL/minimap2/\n",
            "trinityrnaseq-v2.12.0/WDL/minimap2/minimap2_align_to_reference.wdl\n",
            "trinityrnaseq-v2.12.0/WDL/minimap2/build.sh\n",
            "trinityrnaseq-v2.12.0/WDL/minimap2/Dockerfile\n",
            "trinityrnaseq-v2.12.0/LICENSE.txt\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Computing-and-Time-requirements.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-FAQ.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/How-do-I-use-reads-I-downloaded-from-SRA?.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Genome-Structure-Annotation-Using-Trinity--and-PASA.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-Progress-Monitoring.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Genome-Guided-Trinity-Transcriptome-Assembly.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Installing-Trinity.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-On-Terra.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/SuperTranscripts.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Running-Trinity.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Functional-Annotation-of-Transcripts.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Examine-Strand-Specificity.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Home.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/How-can-I-run-this-in-parallel-on-a-computing-grid?.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-Insilico-Normalization.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/DiffTranscriptUsage.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-Transcript-Quantification.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/QC-Samples-and-Biological-Replicates.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Output-of-Trinity-Assembly.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Post-Transcriptome-Assembly-Downstream-Analyses.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-in-Docker.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/There-are-too-many-transcripts!-What-do-I-do?.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/_Sidebar.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-Differential-Expression.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Sample-Specific-Expression.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Errors-during-Trinity-run.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-Runtime-Profiling.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/data/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/data/strand_specificity_analysis_RF.dat\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/data/strand_specificity_SS_but_run_DS.dat\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/data/strand_specificity_analysis_DS.dat\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Transcriptome-Contig-Nx-and-ExN50-stats.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Running-GOSeq.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Contributing.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-Tidbits.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_MA/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_MA/js/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_MA/js/data.js\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_MA/js/glimma.min.js\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_MA/MA-plot.html\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_MA/css/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_MA/css/glimma.min.css\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_volcano/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_volcano/volcano.html\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_volcano/js/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_volcano/js/data.js\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_volcano/js/glimma.min.js\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_volcano/css/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/misc/glimma_volcano/css/glimma.min.css\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/How-do-I-identify-the-specific-reads-that-were-incorporated-into-the-transcript-assemblies?.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/RNA-Seq-Read-Representation-by-Trinity-Assembly.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/How-do-I-combine-reads?.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Transcriptome-Assembly-Quality-Assessment.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/strand_specificity_SS_but_run_DS.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/supertranscript.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/hierarchical_clustering_menu.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/initial_MeV_loaded_data_view.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/set_color_scale_limits_menu.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/change_heatmap_element_size.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/load_data_menu.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/mean_center_data.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/load_data_dialog.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/click_first_data_cell.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/set_color_scale_limits_dialog.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/HCL_heatmap.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/HCL_analysis_tab.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/heatmap_cluster_selected.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/select_matrix_file.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MeV_guide/hierarchical_clustering_dialog.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/nbt_cover.gif\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/tissue_enriched_expression.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MA_and_volcano_plot.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/FL_Spombe_saturation.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/genes_vs_samples_heatmap.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/superTranscripts_VarCalling.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/TrinityCompositeLogo.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/ExN50_plot.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/replicate_pair_correlations.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/PCA.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/sample_correlation_matrix.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/clustered_heatmap.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/collectl_monitoring_mouse50M.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/manually_selected_clusters.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/igv_view_trinity_reads.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/cut_tree_60pct_gene_clusters.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/Nature_Trinity_cov.small.gif\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/IGV_Trinity_screenshot.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/trinity_button.jpg\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/strand_specificity.jpg\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/DGE_DTE_DTU.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/extrapolate_gene_count.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/sample_cor_matrix.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/Ex_vs_N50.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/replicate_pair_MA_plot.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/hcl_genes_tree.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/collectl_monitoring_mouse10M.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/gene_count_vs_min_TPM.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/TrinityGridExec.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/FusionInspectorsvg.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/strand_specificity_analysis_DS.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/genes_vs_minFPKM_plot.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/sum_of_mapped_fragments.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/strand_specificity_analysis_RF.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/MAplotSmearEdgeR.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/FL_mouse_saturation.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/replicate_pair_scatter_plot.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/moby.svg\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/images/expression_profiles_for_clusters.png\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Navigating-Trinity-DE-features-Using-TM4-MeV.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Counting-Full-Length-Trinity-Transcripts.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Coding-Region-Identification-in-Trinity-Assemblies.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Variant-Calling.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Accessing-Trinity-on-Publicly-Available-Compute-Resources.md\n",
            "trinityrnaseq-v2.12.0/trinityrnaseq.wiki/Trinity-Computing-Requirements.md\n",
            "trinityrnaseq-v2.12.0/__pull_trinity_ext_sample_data.sh\n",
            "trinityrnaseq-v2.12.0/.travis.yml\n",
            "trinityrnaseq-v2.12.0/Butterfly/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/README.md\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/lib/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/lib/java-getopt-1.0.13.jar\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/lib/jung-graph-impl-2.0.1.jar\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/lib/Jaligner.jar\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/lib/collections-generic-4.01.jar\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/lib/jung-algorithms-2.0.1.jar\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/lib/jung-api-2.0.1.jar\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/runExample.sh\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/.err\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/.classpath\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/_t_seqB.fa\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/_t_seqA.fa\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/run_butterfly_nonJar.sh\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/README\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/Path_n_MM_count.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/UniquePathContentComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PathReadSupportComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SimpleEdge.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/BFLY_GLOBALS.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PathEndSeqVertexFinishTimeComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/AlignmentStats.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SeqVertex.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/ScoredPath.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/IsoformExpressionLearning.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PathExpressionComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/ZipperAlignment.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PathWithOrig.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/Visitor.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/GraphPath.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SeqVertexFinishTimeComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PairPath.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SimpleEdgeComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/Read.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SeqVertexNodeDepthComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/My_DFS.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/BipartiteMatching.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/DFS.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/NWalign.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/TopologicalSort.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SeqComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/LocInGraph.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PathOverlap.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PasaVertex.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/TransAssembly_allProbPaths.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/util/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/util/Commons.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/util/SequenceParserException.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/util/SequenceParser.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/util/package.html\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/NeedlemanWunschGotohExample.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/NeedlemanWunschExample.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/sequences/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/sequences/p53_human.fasta\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/sequences/p53_mouse.fasta\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/SmithWatermanGotohExample.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/example/package.html\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/MatricesCompartor.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM410\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM80\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM180\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM370\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/EDNAFULL\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM120\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM30\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/IDENTITY\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM50\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM430\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM170\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM210\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM50\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM250\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM90\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM55\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM330\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM75\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM70\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM480\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM110\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUMN\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM260\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM310\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM90\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM60\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM45\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM200\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM360\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM240\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM10\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/GONNET\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM100\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM220\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM460\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM490\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/DAYHOFF\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM230\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM190\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM40\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM85\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM380\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM40\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM70\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM270\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM320\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM350\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM470\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM140\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM20\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM440\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM280\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM100\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM160\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM500\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM65\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM60\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM80\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM390\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM400\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM420\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM30\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM450\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM340\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM150\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM300\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/MATCH\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM35\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM130\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM290\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM62\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/MatrixGenerator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/Matrix.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/package.html\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/MatrixLoaderException.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/matrix/MatrixLoader.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/Cell.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/test/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/test/NeedlemanWunschTester.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/test/NeedlemanWunschGotohTester.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/test/SmithWatermanGotohTester.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/test/package.html\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/test/RandomSequenceGenerator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/TracebackNode.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/Sequence.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/Alignment.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/NeedlemanWunschGotoh.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/Markups.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/Directions.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/NeedlemanWunschGotohBanded.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/SmithWatermanGotoh.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/package.html\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/ui/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/ui/filechooser/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/ui/filechooser/NamedInputStream.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/NeedlemanWunsch.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/formats/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/formats/Format.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/formats/FormatFactory.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/formats/CLUSTAL.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/formats/Pair.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/formats/package.html\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/jaligner/formats/FASTA.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SeqVertexDepthComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/NumPathNodeLoopsEdgeComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/PairPathWOrig.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/SimplePathNodeEdge.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/edu/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/edu/uci/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/edu/uci/ics/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/edu/uci/ics/jung/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/edu/uci/ics/jung/algorithms/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/edu/uci/ics/jung/algorithms/shortestpath/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/edu/uci/ics/jung/algorithms/shortestpath/DijkstraDistanceWoVer.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/ListComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/numLoopsEdgeComparator.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/src/Path.java\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/build.xml\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/run_NWalign.sh\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/sample_data/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/sample_data/c1.graph.out\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/sample_data/c1.graph.reads\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/.settings/\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/.settings/org.eclipse.jdt.core.prefs\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/src/.project\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly/Butterfly.jar\n",
            "trinityrnaseq-v2.12.0/Butterfly/Butterfly.jar\n",
            "trinityrnaseq-v2.12.0/developer.notes\n",
            "trinityrnaseq-v2.12.0/Analysis/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/analyze_diff_expr.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/filter_matrix_min_sum_rowcounts.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/plot_expression_patterns.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/rank_roku_by_expr.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/identify_diff_isoform_splicing.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/extract_GO_enriched_genes.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/subcluster_to_canvasXpress_html.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/plot_all_DE_volcanos.Rscript\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/diff_expr_analysis_to_heatmap_html.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/GOplot.Rscript\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/filter_diff_expr.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/subcluster_to_canvasXpress_html.make_index_html.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/run_TMM_normalization_write_FPKM_matrix.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/remove_batch_effects_from_count_matrix.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/validate_UP_subset.Rscript\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/summarize_diff_expr_across_min_threshold_ranges.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/get_transcript_lengths.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/merge_subclusters.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/rename_matrix_feature_identifiers.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/prune_isoforms_fasta.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/plot_all_DE_MAplots.Rscript\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/assign_tissue_specific.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/PtR\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/add_annot_to_trans_id.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/add_blastx_hit_to_trinity_id.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/stratify_diff_expression.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/plot_log2FC_hist.Rscript\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/matrix_to_gene_plots.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/mmdiff_to_pairwise_summary.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/add_counts_to_classes.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/class_to_separate_fpkm_matrices.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/extract_venn_agree_from_summaries.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/EBSeq_to_pairwise_summary.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/pairwise_DE_summary_to_DE_classification.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/venn_pairwise_summaries.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/examine_rank_correlation.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/notes\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/DE_pair_counts_to_matrix.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/edgeR_to_pairwise_summary.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/pairwise_summaries/venn_pairwise_summaries.pair_stats.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/DTE_to_DTU.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/run_DE_analysis.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/downsample_count_matrix.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/prune_isoforms_gtf.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/rename_matrix_column_labels.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/edgeR.TMM.minimal.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/edgeR_funcs.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/tests/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/tests/test_heatmap_w_pca.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/vioplot2.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/test.heatmap.3.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/pairs3.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/heatmap.3.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/get_cluster_info.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/jaccard_distance.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/manually_define_clusters.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/misc_rnaseq_funcs.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/Glimma.Trinity.Rscript\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/DE_graph_to_dot.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/run_GOseq.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/define_clusters_by_cutting_tree.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cut_tree_into_clusters.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/diff_express.cgi\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/prep_n_run_GOplot.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/replicates_to_sample_averages_matrix.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/Islam_scde_data/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/Islam_scde_data/es.mef.fpkm.matrix\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/MLF_ESC_NPC.cuff.genes.fpkm.matrix.gz\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/runMe.sh\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/samples.txt\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/cleanme.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/cluster_sample_data/orig.samples.txt\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/get_tissue_enriched_DE_one_vs_all.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/add_annotations_to_GO_and_lengths_file.R\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/log2_transform_matrix.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/ROKU.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/log2_transform_median_center_fpkm_matrix.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/TissueEnrichment/\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/TissueEnrichment/README.md\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/TissueEnrichment/group_isoforms_by_tissue_enrichment.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/TissueEnrichment/pairwise_DE_summary_to_DE_classification.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/TissueEnrichment/DE_results_to_pairwise_summary.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/TissueEnrichment/DE_graph_to_dot.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/merge_matrices.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/DifferentialExpression/compare_gene_trans_DE_ranks.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/AllelicVariants/\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/AllelicVariants/util/\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/AllelicVariants/util/clean_bam.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/AllelicVariants/run_variant_calling.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/AllelicVariants/VCF_to_annotated_SNP_report.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/DTU/\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/DTU/util/\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/DTU/util/reformat_featureCounts.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/DTU/README.md\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/DTU/dexseq_wrapper.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/README.md\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/TGraph.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Trinity_fasta_parser.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/TGLOBALS.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Node_alignment.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/TNode.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/GraphCycleException.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Splice_model_refiner.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Node_path.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Compact_graph_partial.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Topological_sort.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/DP_matrix.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Trinity_util.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/__init__.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Compact_graph_whole.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Gene_splice_modeler.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/pylib/Compact_graph_pruner.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/extract_supertranscript_from_reference.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/Trinity_gene_splice_modeler.py\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/_misc/\n",
            "trinityrnaseq-v2.12.0/Analysis/SuperTranscripts/_misc/aln_before_after.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/FL_trans_analysis_pipeline.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/count_by_expression_quintile.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/util/\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/util/blat_full_length_mappings.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/util/blat_map_filter_with_isoforms.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/util/blat_psl_to_align_summary_stats.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/util/blat_top_tier_genes.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/util/blat_query_top_hit_extractor.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/get_genes_from_maps_file.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/tier_gene_trans_alignments.tiers_to_boxplot.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/compute_oracle.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/fusion_comparisons_via_maps_files.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/maps_file_to_paralog_representation.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/R/\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/R/boot.tree.R\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/tier_gene_trans_alignments.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/FL_reconstruction_analysis/oracle_counter.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/GenomeViewPlugin/\n",
            "trinityrnaseq-v2.12.0/Analysis/GenomeViewPlugin/trinity_to_genomeview_html.pl\n",
            "trinityrnaseq-v2.12.0/Analysis/GenomeViewPlugin/test_session.html\n",
            "trinityrnaseq-v2.12.0/Analysis/GenomeViewPlugin/test.html\n",
            "trinityrnaseq-v2.12.0/sample_data/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/reads2.right.fq.gz\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/longReads.fa\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/samples.PE.txt\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/samples.SE.txt\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/test_FL.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/Makefile\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/reads.right.fa.gz\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/README\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/refSeqs.fa\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.IGV.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/region7.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/ex7.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/example7.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/ex7.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/region7.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/ex07.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/region3.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/example3.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/ex3.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/ex3.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/ex03.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/region3.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/ex05.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/ex5.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta.gmap.gff\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/example5.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta.Bfly.dot\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/runMe.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta.gmap.bed\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/region5.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/clean.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/ex5.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/runMe.clean.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/clean.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/region5.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/align_reads_via_bowtie.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/ex2.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/ex02.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/region2.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/region2.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/ex2.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/example2.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/refSeqs.fa.gene_trans_map\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/example6.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/region6.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/region6.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/ex6.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/ex06.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/ex6.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/region8.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/region8.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/ex8.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/ex8.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/example8.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/ex08.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/bowtie2.sam.aligned.bam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/region4.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/ex4.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/ex4.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/example4.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/ex04.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/region4.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/ex9.reads.right.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/region9.sam.adj\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/region9.sam\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/runMe.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/ex09.refSeq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/ex9.reads.left.fq\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/example9.png\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/runMe.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/reads2.left.fq.gz\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/reads.left.fa.gz\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/cleanme.pl\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/reads.right.fq.gz\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/reads.left.fq.gz\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_docker.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_PasaFly.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_lenient_path_extension.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__run_PE_samples_file.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_piecemeal.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_para_iworm.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe.singleFQ.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__run_SE_samples_file.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_as_DS.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_use_my_ramdisk.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_full_cleanup.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_with_qual_trimming.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_include_normalization.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_parallel_iworm_assembly.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_wSuperReads.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/runMe_no_qualTrim.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_CuffFly.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_use_bowtie2.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_using_Grid_SGE.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_cleanup.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_singularity.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_using_Grid_uger.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_w_monitoring.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe.noSeqtk.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_docker.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_using_Grid_LSF.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_bowtie_components.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_with_qual_trimming_and_normalize_libs_separately.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__run_PE_samples_file.min_kmer_cov_3.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_use_workdir.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe.diff_kmer_length.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_salmon_no_cleanup.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_trin_complete.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__run_abundance_estimation_include_antisense.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_include_supertranscripts.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_include_long_reads.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_NO_normalization.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_with_jaccard_clip.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_with_qual_trimming_and_normalization.sh\n",
            "trinityrnaseq-v2.12.0/sample_data/Makefile\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/meta.yaml\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/trinity-plugins.makefile.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.6.6/\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.6.6/trinity-plugins.makefile.clean.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.6.6/makefile.clean.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.6.6/meta.yaml\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.6.6/build.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.6.6/run_test.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/build.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/chrysalis.bioconda.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.3.2/\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.3.2/Build.PL\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.3.2/trinity-plugins.makefile.clean.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.3.2/makefile.clean.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.3.2/meta.yaml\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.3.2/build.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/run_test.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.1.1/\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.1.1/meta.yaml\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.1.1/build.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.2.0/\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.2.0/Build.PL\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.2.0/trinity-plugins.makefile.clean.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.2.0/makefile.clean.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.2.0/meta.yaml\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/2.2.0/build.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/date.2011_11_26/\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/date.2011_11_26/meta.yaml\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/date.2011_11_26/build.sh\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/date.2011_11_26/samtools_makefile.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/date.2011_11_26/Trinity.pl.patch\n",
            "trinityrnaseq-v2.12.0/bioconda_recipe/trinity/date.2011_11_26/RunButterfly.cc.patch\n",
            "mv: cannot move 'trinityrnaseq-v2.12.0/' to '/usr/local/trinityrnaseq-v2.12.0': Directory not empty\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!apt-get install seqtk\n",
        "!samtools faidx /content/sequence.fasta\n",
        "!wgsim -N 10000 -1 150 -2 150 /content/sequence.fasta /content/left.fq /content/right.fq\n",
        "\n",
        "!seqtk seq -Q64 -V -s 100 /content/left.fq > /content/left_with_qual.fq\n",
        "!seqtk seq -Q64 -V -s 100 /content/right.fq > /content/right_with_qual.fq"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2EXIPKTdTACL",
        "outputId": "8fb98a01-b2e0-4eae-df2d-f7ca6105679d"
      },
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "The following NEW packages will be installed:\n",
            "  seqtk\n",
            "0 upgraded, 1 newly installed, 0 to remove and 16 not upgraded.\n",
            "Need to get 30.2 kB of archives.\n",
            "After this operation, 85.0 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 seqtk amd64 1.3-2 [30.2 kB]\n",
            "Fetched 30.2 kB in 0s (115 kB/s)\n",
            "Selecting previously unselected package seqtk.\n",
            "(Reading database ... 122529 files and directories currently installed.)\n",
            "Preparing to unpack .../archives/seqtk_1.3-2_amd64.deb ...\n",
            "Unpacking seqtk (1.3-2) ...\n",
            "Setting up seqtk (1.3-2) ...\n",
            "Processing triggers for man-db (2.10.2-1) ...\n",
            "[wgsim] seed = 1693701033\n",
            "[wgsim_core] calculating the total length of the reference sequence...\n",
            "[wgsim_core] 1 sequences, total length: 113064\n",
            "gi|902687128|ref|NC_027618.1|\t56\tC\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t1488\tC\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t1760\tA\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t2581\tT\tA\t-\n",
            "gi|902687128|ref|NC_027618.1|\t2901\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t4633\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t5977\tT\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t6468\tT\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t6784\tC\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t7448\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t7705\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t7934\tT\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t8768\t-\tC\t+\n",
            "gi|902687128|ref|NC_027618.1|\t9168\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t9444\tT\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t10208\tC\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t11586\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t11900\tA\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t13234\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t13356\tA\t-\t-\n",
            "gi|902687128|ref|NC_027618.1|\t15098\tA\t-\t-\n",
            "gi|902687128|ref|NC_027618.1|\t15388\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t15778\tC\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t15780\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t18946\tT\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t18962\tA\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t19096\tT\tA\t-\n",
            "gi|902687128|ref|NC_027618.1|\t19814\tT\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t21015\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t21523\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t21966\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t22588\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t24878\tT\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t25619\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t25842\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t28679\tT\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t30405\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t30483\tC\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t31283\tA\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t31330\t-\tC\t+\n",
            "gi|902687128|ref|NC_027618.1|\t31603\t-\tG\t+\n",
            "gi|902687128|ref|NC_027618.1|\t32397\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t34434\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t35736\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t35930\tT\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t35989\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t36516\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t36823\tA\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t37794\tG\t-\t-\n",
            "gi|902687128|ref|NC_027618.1|\t38100\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t40130\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t40766\tC\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t40860\tT\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t41390\tG\t-\t-\n",
            "gi|902687128|ref|NC_027618.1|\t43088\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t43597\tG\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t43832\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t44995\tC\tA\t-\n",
            "gi|902687128|ref|NC_027618.1|\t45488\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t47265\tT\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t50443\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t51735\tC\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t51854\t-\tA\t+\n",
            "gi|902687128|ref|NC_027618.1|\t52162\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t52631\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t52733\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t53011\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t54255\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t54614\tT\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t54922\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t55715\tA\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t56241\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t56718\tT\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t57615\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t57689\tT\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t58388\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t59975\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t59979\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t60229\tA\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t62058\tT\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t62079\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t62285\tC\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t62459\tT\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t63447\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t65163\tG\t-\t-\n",
            "gi|902687128|ref|NC_027618.1|\t65319\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t65566\tG\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t65661\tC\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t66462\tT\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t67248\tG\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t68701\t-\tC\t+\n",
            "gi|902687128|ref|NC_027618.1|\t69016\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t69792\tT\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t72408\tA\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t72877\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t73304\tT\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t74139\t-\tTA\t+\n",
            "gi|902687128|ref|NC_027618.1|\t74376\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t74930\tC\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t74985\tA\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t75661\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t76799\tA\tC\t-\n",
            "gi|902687128|ref|NC_027618.1|\t77512\tT\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t78009\tC\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t79007\tC\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t79987\tG\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t85282\tG\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t86948\tA\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t87773\tT\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t89370\t-\tGA\t-\n",
            "gi|902687128|ref|NC_027618.1|\t89935\tA\tG\t-\n",
            "gi|902687128|ref|NC_027618.1|\t95291\tC\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t95386\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t95489\tC\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t96332\tT\tA\t-\n",
            "gi|902687128|ref|NC_027618.1|\t96370\tG\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t97461\tG\tK\t+\n",
            "gi|902687128|ref|NC_027618.1|\t98077\tT\tA\t-\n",
            "gi|902687128|ref|NC_027618.1|\t99609\tAATTTT\t-\t-\n",
            "gi|902687128|ref|NC_027618.1|\t99855\tC\tY\t+\n",
            "gi|902687128|ref|NC_027618.1|\t100663\tA\tR\t+\n",
            "gi|902687128|ref|NC_027618.1|\t100993\tT\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t101440\tA\tT\t-\n",
            "gi|902687128|ref|NC_027618.1|\t101489\tT\tW\t+\n",
            "gi|902687128|ref|NC_027618.1|\t102739\tG\tA\t-\n",
            "gi|902687128|ref|NC_027618.1|\t106393\tG\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t111830\tC\tS\t+\n",
            "gi|902687128|ref|NC_027618.1|\t112141\tC\tM\t+\n",
            "gi|902687128|ref|NC_027618.1|\t112419\tA\tC\t-\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mkdir /content/my_trinity_output\n",
        "\n",
        "!/usr/local/trinityrnaseq-v2.12.0/Trinity --seqType fq --left /content/left.fq --right /content/right.fq --max_memory 10G --CPU 2 --output /content/my_trinity_output\n",
        "\n",
        "#!/content/trinityrnaseq-v2.15.1/Trinity --seqType fq --single /content/sequence.fasta --max_memory 10G --CPU 2 --output trinity_output\n",
        "\n",
        "!zip -r /content/archive_name.zip /content/my_trinity_output\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "pc1OE1KjNro9",
        "outputId": "82de0bc8-6b3f-4416-f94f-a79d84b825d8"
      },
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "mkdir: cannot create directory ‘/content/my_trinity_output’: File exists\n",
            "\n",
            "\n",
            "     ______  ____   ____  ____   ____  ______  __ __\n",
            "    |      ||    \\ |    ||    \\ |    ||      ||  |  |\n",
            "    |      ||  D  ) |  | |  _  | |  | |      ||  |  |\n",
            "    |_|  |_||    /  |  | |  |  | |  | |_|  |_||  ~  |\n",
            "      |  |  |    \\  |  | |  |  | |  |   |  |  |___, |\n",
            "      |  |  |  .  \\ |  | |  |  | |  |   |  |  |     |\n",
            "      |__|  |__|\\_||____||__|__||____|  |__|  |____/\n",
            "\n",
            "    Trinity-v2.12.0\n",
            "\n",
            "\n",
            "\n",
            "Left read files: $VAR1 = [\n",
            "          '/content/left.fq'\n",
            "        ];\n",
            "Right read files: $VAR1 = [\n",
            "          '/content/right.fq'\n",
            "        ];\n",
            "Trinity version: Trinity-v2.12.0\n",
            "** NOTE: Latest version of Trinity is Trinity-v2.15.1, and can be obtained at:\n",
            "\thttps://github.com/trinityrnaseq/trinityrnaseq/releases\n",
            "\n",
            "Sunday, September 3, 2023: 00:34:52\tCMD: java -Xmx64m -XX:ParallelGCThreads=2  -jar /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/ExitTester.jar 0\n",
            "Sunday, September 3, 2023: 00:34:52\tCMD: java -Xmx4g -XX:ParallelGCThreads=2  -jar /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/ExitTester.jar 1\n",
            "Sunday, September 3, 2023: 00:34:53\tCMD: mkdir -p /content/my_trinity_output/chrysalis\n",
            "\n",
            "\n",
            "----------------------------------------------------------------------------------\n",
            "-------------- Trinity Phase 1: Clustering of RNA-Seq Reads  ---------------------\n",
            "----------------------------------------------------------------------------------\n",
            "\n",
            "---------------------------------------------------------------\n",
            "------------ In silico Read Normalization ---------------------\n",
            "-- (Removing Excess Reads Beyond 200 Coverage --\n",
            "---------------------------------------------------------------\n",
            "\n",
            "# running normalization on reads: $VAR1 = [\n",
            "          [\n",
            "            '/content/left.fq'\n",
            "          ],\n",
            "          [\n",
            "            '/content/right.fq'\n",
            "          ]\n",
            "        ];\n",
            "\n",
            "\n",
            "Sunday, September 3, 2023: 00:34:53\tCMD: /usr/local/trinityrnaseq-v2.12.0/util/insilico_read_normalization.pl --seqType fq --JM 10G  --max_cov 200 --min_cov 1 --CPU 2 --output /content/my_trinity_output/insilico_read_normalization --max_CV 10000  --left /content/left.fq --right /content/right.fq --pairs_together  --PARALLEL_STATS  \n",
            "-prepping seqs\n",
            "Converting input files. (both directions in parallel)CMD: seqtk-trinity seq -A -R 1  /content/left.fq >> left.fa\n",
            "CMD: seqtk-trinity seq -A -R 2  /content/right.fq >> right.fa\n",
            "CMD finished (0 seconds)\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch left.fa.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch right.fa.ok\n",
            "CMD finished (0 seconds)\n",
            "Done converting input files.CMD: cat left.fa right.fa > both.fa\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch both.fa.ok\n",
            "CMD finished (0 seconds)\n",
            "-kmer counting.\n",
            "-------------------------------------------\n",
            "----------- Jellyfish  --------------------\n",
            "-- (building a k-mer catalog from reads) --\n",
            "-------------------------------------------\n",
            "\n",
            "CMD: jellyfish count -t 2 -m 25 -s 100000000  --canonical  both.fa\n",
            "CMD finished (4 seconds)\n",
            "CMD: jellyfish histo -t 2 -o jellyfish.K25.min2.kmers.fa.histo mer_counts.jf\n",
            "CMD finished (0 seconds)\n",
            "CMD: jellyfish dump -L 2 mer_counts.jf > jellyfish.K25.min2.kmers.fa\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch jellyfish.K25.min2.kmers.fa.success\n",
            "CMD finished (0 seconds)\n",
            "-generating stats files\n",
            "CMD: /usr/local/trinityrnaseq-v2.12.0/util/..//Inchworm/bin/fastaToKmerCoverageStats --reads left.fa --kmers jellyfish.K25.min2.kmers.fa --kmer_size 25  --num_threads 1  --DS  > left.fa.K25.stats\n",
            "CMD: /usr/local/trinityrnaseq-v2.12.0/util/..//Inchworm/bin/fastaToKmerCoverageStats --reads right.fa --kmers jellyfish.K25.min2.kmers.fa --kmer_size 25  --num_threads 1  --DS  > right.fa.K25.stats\n",
            "-reading Kmer occurrences...\n",
            "-reading Kmer occurrences...\n",
            "\n",
            " done parsing 162857 Kmers, 162857 added, taking 1 seconds.\n",
            "\n",
            " done parsing 162857 Kmers, 162857 added, taking 1 seconds.\n",
            "STATS_GENERATION_TIME: 0 seconds.\n",
            "CMD finished (1 seconds)\n",
            "STATS_GENERATION_TIME: 0 seconds.\n",
            "CMD finished (1 seconds)\n",
            "CMD: touch left.fa.K25.stats.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch right.fa.K25.stats.ok\n",
            "CMD finished (0 seconds)\n",
            "-sorting each stats file by read name.\n",
            "CMD: head -n1 left.fa.K25.stats > left.fa.K25.stats.sort && tail -n +2 left.fa.K25.stats | /usr/bin/sort --parallel=2 -k1,1 -T . -S 5G >> left.fa.K25.stats.sort\n",
            "CMD: head -n1 right.fa.K25.stats > right.fa.K25.stats.sort && tail -n +2 right.fa.K25.stats | /usr/bin/sort --parallel=2 -k1,1 -T . -S 5G >> right.fa.K25.stats.sort\n",
            "CMD finished (0 seconds)\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch left.fa.K25.stats.sort.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch right.fa.K25.stats.sort.ok\n",
            "CMD finished (0 seconds)\n",
            "-defining normalized reads\n",
            "CMD: /usr/local/trinityrnaseq-v2.12.0/util/..//util/support_scripts//nbkc_merge_left_right_stats.pl --left left.fa.K25.stats.sort --right right.fa.K25.stats.sort --sorted > pairs.K25.stats\n",
            "-opening left.fa.K25.stats.sort\n",
            "-opening right.fa.K25.stats.sort\n",
            "-done opening files.\n",
            "CMD finished (1 seconds)\n",
            "CMD: touch pairs.K25.stats.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: /usr/local/trinityrnaseq-v2.12.0/util/..//util/support_scripts//nbkc_normalize.pl --stats_file pairs.K25.stats --max_cov 200  --min_cov 1 --max_CV 10000 > pairs.K25.stats.C200.maxCV10000.accs\n",
            "10000 / 10000 = 100.00% reads selected during normalization.\n",
            "0 / 10000 = 0.00% reads discarded as likely aberrant based on coverage profiles.\n",
            "0 / 10000 = 0.00% reads discarded as below minimum coverage threshold=1\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch pairs.K25.stats.C200.maxCV10000.accs.ok\n",
            "CMD finished (0 seconds)\n",
            "-search and capture.\n",
            "-preparing to extract selected reads from: /content/left.fq ... done prepping, now search and capture.\n",
            "-capturing normalized reads from: /content/left.fq\n",
            "-preparing to extract selected reads from: /content/right.fq ... done prepping, now search and capture.\n",
            "-capturing normalized reads from: /content/right.fq\n",
            "CMD: touch /content/my_trinity_output/insilico_read_normalization/left.fq.normalized_K25_maxC200_minC1_maxCV10000.fq.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch /content/my_trinity_output/insilico_read_normalization/right.fq.normalized_K25_maxC200_minC1_maxCV10000.fq.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: ln -sf /content/my_trinity_output/insilico_read_normalization/left.fq.normalized_K25_maxC200_minC1_maxCV10000.fq left.norm.fq\n",
            "CMD finished (0 seconds)\n",
            "CMD: ln -sf /content/my_trinity_output/insilico_read_normalization/right.fq.normalized_K25_maxC200_minC1_maxCV10000.fq right.norm.fq\n",
            "CMD finished (0 seconds)\n",
            "-removing tmp dir /content/my_trinity_output/insilico_read_normalization/tmp_normalized_reads\n",
            "\n",
            "\n",
            "Normalization complete. See outputs: \n",
            "\t/content/my_trinity_output/insilico_read_normalization/left.fq.normalized_K25_maxC200_minC1_maxCV10000.fq\n",
            "\t/content/my_trinity_output/insilico_read_normalization/right.fq.normalized_K25_maxC200_minC1_maxCV10000.fq\n",
            "Sunday, September 3, 2023: 00:34:59\tCMD: touch /content/my_trinity_output/insilico_read_normalization/normalization.ok\n",
            "Converting input files. (in parallel)Sunday, September 3, 2023: 00:34:59\tCMD: cat /content/my_trinity_output/insilico_read_normalization/left.norm.fq | seqtk-trinity seq -A -R 1 - >> left.fa\n",
            "Sunday, September 3, 2023: 00:34:59\tCMD: cat /content/my_trinity_output/insilico_read_normalization/right.norm.fq | seqtk-trinity seq -A -R 2 - >> right.fa\n",
            "Sunday, September 3, 2023: 00:34:59\tCMD: touch left.fa.ok\n",
            "Sunday, September 3, 2023: 00:34:59\tCMD: touch right.fa.ok\n",
            "Sunday, September 3, 2023: 00:34:59\tCMD: touch left.fa.ok right.fa.ok\n",
            "Sunday, September 3, 2023: 00:34:59\tCMD: cat left.fa right.fa > /content/my_trinity_output/both.fa\n",
            "Sunday, September 3, 2023: 00:34:59\tCMD: touch /content/my_trinity_output/both.fa.ok\n",
            "-------------------------------------------\n",
            "----------- Jellyfish  --------------------\n",
            "-- (building a k-mer (25) catalog from reads) --\n",
            "-------------------------------------------\n",
            "\n",
            "* [Sun Sep  3 00:34:59 2023] Running CMD: jellyfish count -t 2 -m 25 -s 100000000 -o mer_counts.25.asm.jf --canonical  /content/my_trinity_output/both.fa\n",
            "* [Sun Sep  3 00:35:02 2023] Running CMD: jellyfish dump -L 1 mer_counts.25.asm.jf > jellyfish.kmers.25.asm.fa\n",
            "* [Sun Sep  3 00:35:02 2023] Running CMD: jellyfish histo -t 2 -o jellyfish.kmers.25.asm.fa.histo mer_counts.25.asm.jf\n",
            "----------------------------------------------\n",
            "--------------- Inchworm (K=25, asm) ---------------------\n",
            "-- (Linear contig construction from k-mers) --\n",
            "----------------------------------------------\n",
            "\n",
            "* [Sun Sep  3 00:35:02 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/Inchworm/bin//inchworm --kmers jellyfish.kmers.25.asm.fa --run_inchworm -K 25 --monitor 1   --DS  --num_threads 2  --PARALLEL_IWORM   --min_any_entropy 1.0   -L 25  --no_prune_error_kmers  > /content/my_trinity_output/inchworm.DS.fa.tmp\n",
            "Kmer length set to: 25\n",
            "Min assembly length set to: 25\n",
            "Monitor turned on, set to: 1\n",
            "double stranded mode set\n",
            "min entropy set to: 1\n",
            "setting number of threads to: 2\n",
            "-setting parallel iworm mode.\n",
            "-reading Kmer occurrences...\n",
            " [0M] Kmers parsed.     \n",
            " done parsing 1066666 Kmers, 1066666 added, taking 2 seconds.\n",
            "\n",
            "TIMING KMER_DB_BUILDING 2 s.\n",
            "Pruning kmers (min_kmer_count=1 min_any_entropy=1 min_ratio_non_error=0.005)\n",
            "Pruned 609 kmers from catalog.\n",
            "\tPruning time: 0 seconds = 0 minutes.\n",
            "\n",
            "TIMING PRUNING 0 s.\n",
            "-populating the kmer seed candidate list.\n",
            "Kcounter hash size: 1066666\n",
            "Processed 1066057 non-zero abundance kmers in kcounter.\n",
            "-Not sorting list of kmers, given parallel mode in effect.\n",
            "-beginning inchworm contig assembly.\n",
            "Total kcounter hash size: 1066666 vs. sorted list size: 1066057\n",
            "num threads set to: 2\n",
            "Done opening file. tmp.iworm.fa.pid_16184.thread_0\n",
            "Done opening file. tmp.iworm.fa.pid_16184.thread_1\n",
            "\n",
            "\tIworm contig assembly time: 5 seconds = 0.0833333 minutes.\n",
            "\n",
            "TIMING CONTIG_BUILDING 5 s.\n",
            "\n",
            "TIMING PROG_RUNTIME 7 s.\n",
            "* [Sun Sep  3 00:35:10 2023] Running CMD: mv /content/my_trinity_output/inchworm.DS.fa.tmp /content/my_trinity_output/inchworm.DS.fa\n",
            "Sunday, September 3, 2023: 00:35:10\tCMD: touch /content/my_trinity_output/inchworm.DS.fa.finished\n",
            "--------------------------------------------------------\n",
            "-------------------- Chrysalis -------------------------\n",
            "-- (Contig Clustering & de Bruijn Graph Construction) --\n",
            "--------------------------------------------------------\n",
            "\n",
            "inchworm_target: /content/my_trinity_output/both.fa\n",
            "bowtie_reads_fa: /content/my_trinity_output/both.fa\n",
            "chrysalis_reads_fa: /content/my_trinity_output/both.fa\n",
            "* [Sun Sep  3 00:35:10 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/filter_iworm_by_min_length_or_cov.pl /content/my_trinity_output/inchworm.DS.fa 100 10 > /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100\n",
            "* [Sun Sep  3 00:35:10 2023] Running CMD: /usr/bin/bowtie2-build --threads 2 -o 3 /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100 /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100 1>/dev/null\n",
            "* [Sun Sep  3 00:35:10 2023] Running CMD: bash -c \" set -o pipefail;/usr/bin/bowtie2 --local -k 2 --no-unal --threads 2 -f --score-min G,20,8 -x /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100 /content/my_trinity_output/both.fa  | samtools view -@ 2 -F4 -Sb - | samtools sort -m 2684354560 -@ 2 -no - - > /content/my_trinity_output/chrysalis/iworm.bowtie.nameSorted.bam\" \n",
            "* [Sun Sep  3 00:35:21 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/scaffold_iworm_contigs.pl /content/my_trinity_output/chrysalis/iworm.bowtie.nameSorted.bam /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100 > /content/my_trinity_output/chrysalis/iworm_scaffolds.txt\n",
            "* [Sun Sep  3 00:35:22 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/Chrysalis/bin/GraphFromFasta -i /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100 -r /content/my_trinity_output/both.fa -min_contig_length 200 -min_glue 2 -glue_factor 0.05 -min_iso_ratio 0.05 -t 2 -k 24 -kk 48  -scaffolding /content/my_trinity_output/chrysalis/iworm_scaffolds.txt  > /content/my_trinity_output/chrysalis/iworm_cluster_welds_graph.txt\n",
            "* [Sun Sep  3 00:35:26 2023] Running CMD: /usr/bin/sort --parallel=2 -T . -S 10G  -k9,9gr /content/my_trinity_output/chrysalis/iworm_cluster_welds_graph.txt > /content/my_trinity_output/chrysalis/iworm_cluster_welds_graph.txt.sorted\n",
            "* [Sun Sep  3 00:35:26 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/annotate_chrysalis_welds_with_iworm_names.pl /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100 /content/my_trinity_output/chrysalis/iworm_cluster_welds_graph.txt.sorted > /content/my_trinity_output/chrysalis/iworm_cluster_welds_graph.txt.sorted.wIwormNames\n",
            "* [Sun Sep  3 00:35:26 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/Chrysalis/bin/BubbleUpClustering -i /content/my_trinity_output/chrysalis/inchworm.DS.fa.min100  -weld_graph /content/my_trinity_output/chrysalis/iworm_cluster_welds_graph.txt.sorted -min_contig_length 200 -max_cluster_size 25  > /content/my_trinity_output/chrysalis/GraphFromIwormFasta.out\n",
            "* [Sun Sep  3 00:35:26 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/Chrysalis/bin/CreateIwormFastaBundle -i /content/my_trinity_output/chrysalis/GraphFromIwormFasta.out -o /content/my_trinity_output/chrysalis/bundled_iworm_contigs.fasta -min 200\n",
            "* [Sun Sep  3 00:35:26 2023] Running CMD: /usr/local/trinityrnaseq-v2.12.0/Chrysalis/bin/ReadsToTranscripts -i /content/my_trinity_output/both.fa -f /content/my_trinity_output/chrysalis/bundled_iworm_contigs.fasta -o /content/my_trinity_output/chrysalis/readsToComponents.out -t 2 -max_mem_reads 50000000  -p 10 \n",
            "* [Sun Sep  3 00:35:36 2023] Running CMD: /usr/bin/sort --parallel=2 -T . -S 10G -k 1,1n /content/my_trinity_output/chrysalis/readsToComponents.out > /content/my_trinity_output/chrysalis/readsToComponents.out.sort\n",
            "Sunday, September 3, 2023: 00:35:36\tCMD: mkdir -p /content/my_trinity_output/read_partitions/Fb_0/CBin_0\n",
            "Sunday, September 3, 2023: 00:35:36\tCMD: touch /content/my_trinity_output/partitioned_reads.files.list.ok\n",
            "Sunday, September 3, 2023: 00:35:36\tCMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/write_partitioned_trinity_cmds.pl --reads_list_file /content/my_trinity_output/partitioned_reads.files.list --CPU 1 --max_memory 1G  --run_as_paired  --seqType fa --trinity_complete --full_cleanup  > recursive_trinity.cmds\n",
            "Sunday, September 3, 2023: 00:35:36\tCMD: touch recursive_trinity.cmds.ok\n",
            "Sunday, September 3, 2023: 00:35:36\tCMD: touch recursive_trinity.cmds.ok\n",
            "\n",
            "\n",
            "--------------------------------------------------------------------------------\n",
            "------------ Trinity Phase 2: Assembling Clusters of Reads ---------------------\n",
            "------- (involving the Inchworm, Chrysalis, Butterfly trifecta ) ---------------\n",
            "--------------------------------------------------------------------------------\n",
            "\n",
            "Sunday, September 3, 2023: 00:35:37\tCMD: /usr/local/trinityrnaseq-v2.12.0/trinity-plugins/BIN/ParaFly -c recursive_trinity.cmds -CPU 2 -v -shuffle \n",
            "Number of Commands: 3\n",
            "\n",
            "\n",
            "Error encountered::  <!----\n",
            "CMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16423.1693701358.stderr\n",
            "\n",
            "Errmsg:\n",
            "CMD: salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.16706.stderr 2>&1\n",
            "Error, cmd: salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.16706.stderr 2>&1 failed with msg: Usage: salmon [options]\n",
            "\n",
            "salmon: error: no such option: --no-version-check\n",
            " Error, cmd:\n",
            "salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.16706.stderr 2>&1\n",
            " died with ret (512) at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../PerlLib/Process_cmd.pm line 19.\n",
            "\tProcess_cmd::process_cmd(\"salmon --no-version-check index -t /content/my_trinity_output\"...) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 41\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 40\n",
            "\tmain::run_cmd_capture_stderr(\"salmon --no-version-check index -t /content/my_trinity_output\"..., \"_salmon.16706.stderr\") called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 24\n",
            "\n",
            "--->\n",
            "\n",
            "-salmon error reported: Error, cmd: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16423.1693701358.stderr died with ret 65280  at /usr/local/trinityrnaseq-v2.12.0/PerlLib/Pipeliner.pm line 187.\n",
            "\tPipeliner::run(Pipeliner=HASH(0x58cf0d0c59a8)) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1907\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1906\n",
            "\tmain::run_Trinity() called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1416\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1415\n",
            "WARNING - salmon failure mode not recognized by Trinity:\n",
            "Error, cmd: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c2.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16423.1693701358.stderr died with ret 65280  at /usr/local/trinityrnaseq-v2.12.0/PerlLib/Pipeliner.pm line 187.\n",
            "\tPipeliner::run(Pipeliner=HASH(0x58cf0d0c59a8)) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1907\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1906\n",
            "\tmain::run_Trinity() called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1416\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1415\n",
            "\n",
            " - retaining Trinity transcripts provided as input to salmon, w/o filtering (pre-salmon mode).\n",
            "succeeded(1)   33.3333% completed.    \n",
            "\n",
            "Error encountered::  <!----\n",
            "CMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16726.1693701402.stderr\n",
            "\n",
            "Errmsg:\n",
            "CMD: salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.17092.stderr 2>&1\n",
            "Error, cmd: salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.17092.stderr 2>&1 failed with msg: Usage: salmon [options]\n",
            "\n",
            "salmon: error: no such option: --no-version-check\n",
            " Error, cmd:\n",
            "salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.17092.stderr 2>&1\n",
            " died with ret (512) at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../PerlLib/Process_cmd.pm line 19.\n",
            "\tProcess_cmd::process_cmd(\"salmon --no-version-check index -t /content/my_trinity_output\"...) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 41\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 40\n",
            "\tmain::run_cmd_capture_stderr(\"salmon --no-version-check index -t /content/my_trinity_output\"..., \"_salmon.17092.stderr\") called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 24\n",
            "\n",
            "--->\n",
            "\n",
            "-salmon error reported: Error, cmd: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16726.1693701402.stderr died with ret 65280  at /usr/local/trinityrnaseq-v2.12.0/PerlLib/Pipeliner.pm line 187.\n",
            "\tPipeliner::run(Pipeliner=HASH(0x5986f68a3330)) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1907\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1906\n",
            "\tmain::run_Trinity() called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1416\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1415\n",
            "WARNING - salmon failure mode not recognized by Trinity:\n",
            "Error, cmd: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c1.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16726.1693701402.stderr died with ret 65280  at /usr/local/trinityrnaseq-v2.12.0/PerlLib/Pipeliner.pm line 187.\n",
            "\tPipeliner::run(Pipeliner=HASH(0x5986f68a3330)) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1907\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1906\n",
            "\tmain::run_Trinity() called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1416\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1415\n",
            "\n",
            " - retaining Trinity transcripts provided as input to salmon, w/o filtering (pre-salmon mode).\n",
            "succeeded(2)   66.6667% completed.    \n",
            "\n",
            "Error encountered::  <!----\n",
            "CMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16422.1693701563.stderr\n",
            "\n",
            "Errmsg:\n",
            "CMD: salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.17806.stderr 2>&1\n",
            "Error, cmd: salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.17806.stderr 2>&1 failed with msg: Usage: salmon [options]\n",
            "\n",
            "salmon: error: no such option: --no-version-check\n",
            " Error, cmd:\n",
            "salmon --no-version-check index -t /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta -i /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta.salmon.idx -k 25 -p 1  > _salmon.17806.stderr 2>&1\n",
            " died with ret (512) at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../PerlLib/Process_cmd.pm line 19.\n",
            "\tProcess_cmd::process_cmd(\"salmon --no-version-check index -t /content/my_trinity_output\"...) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 41\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 40\n",
            "\tmain::run_cmd_capture_stderr(\"salmon --no-version-check index -t /content/my_trinity_output\"..., \"_salmon.17806.stderr\") called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl line 24\n",
            "\n",
            "--->\n",
            "\n",
            "-salmon error reported: Error, cmd: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16422.1693701563.stderr died with ret 65280  at /usr/local/trinityrnaseq-v2.12.0/PerlLib/Pipeliner.pm line 187.\n",
            "\tPipeliner::run(Pipeliner=HASH(0x5c2c99a77430)) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1907\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1906\n",
            "\tmain::run_Trinity() called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1416\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1415\n",
            "WARNING - salmon failure mode not recognized by Trinity:\n",
            "Error, cmd: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/salmon_runner.pl /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/Trinity.tmp.fasta /content/my_trinity_output/read_partitions/Fb_0/CBin_0/c0.trinity.reads.fa.out/single.fa.SR_supp.fa 2>tmp.16422.1693701563.stderr died with ret 65280  at /usr/local/trinityrnaseq-v2.12.0/PerlLib/Pipeliner.pm line 187.\n",
            "\tPipeliner::run(Pipeliner=HASH(0x5c2c99a77430)) called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1907\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1906\n",
            "\tmain::run_Trinity() called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1416\n",
            "\teval {...} called at /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/../../Trinity line 1415\n",
            "\n",
            " - retaining Trinity transcripts provided as input to salmon, w/o filtering (pre-salmon mode).\n",
            "succeeded(3)   100% completed.    \n",
            "\n",
            "All commands completed successfully. :-)\n",
            "\n",
            "\n",
            "\n",
            "** Harvesting all assembled transcripts into a single multi-fasta file...\n",
            "\n",
            "Sunday, September 3, 2023: 00:39:34\tCMD: find /content/my_trinity_output/read_partitions/ -name '*inity.fasta'  | /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/partitioned_trinity_aggregator.pl --token_prefix TRINITY_DN --output_prefix /content/my_trinity_output/Trinity.tmp\n",
            "-relocating Trinity.tmp.fasta to /content/my_trinity_output/Trinity.fasta\n",
            "Sunday, September 3, 2023: 00:39:34\tCMD: mv Trinity.tmp.fasta /content/my_trinity_output/Trinity.fasta\n",
            "\n",
            "\n",
            "###################################################################\n",
            "Trinity assemblies are written to /content/my_trinity_output/Trinity.fasta\n",
            "###################################################################\n",
            "\n",
            "\n",
            "Sunday, September 3, 2023: 00:39:34\tCMD: /usr/local/trinityrnaseq-v2.12.0/util/support_scripts/get_Trinity_gene_to_trans_map.pl /content/my_trinity_output/Trinity.fasta > /content/my_trinity_output/Trinity.fasta.gene_trans_map\n"
          ]
        }
      ]
    }
  ]
}